diff --git a/generate_tiles.py b/generate_tiles.py
index dfa595c..8136b69 100755
--- a/generate_tiles.py
+++ b/generate_tiles.py
@@ -191,66 +191,16 @@ def render_tiles(bbox, mapfile, tile_dir, minZoom=1,maxZoom=18, name="unknown",
 
 
 if __name__ == "__main__":
-    home = os.environ['HOME']
-    try:
-        mapfile = os.environ['MAPNIK_MAP_FILE']
-    except KeyError:
-        mapfile = home + "/svn.openstreetmap.org/applications/rendering/mapnik/osm-local.xml"
+    # mapfile = "elevation.xml"
+    mapfile= sys.argv[1]
+    tile_dir = "tiles/"
+
+    # E, S, W, N, minz, maxz
     try:
-        tile_dir = os.environ['MAPNIK_TILE_DIR']
-    except KeyError:
-        tile_dir = home + "/osm/tiles/"
-
-    if not tile_dir.endswith('/'):
-        tile_dir = tile_dir + '/'
-
-    #-------------------------------------------------------------------------
-    #
-    # Change the following for different bounding boxes and zoom levels
-    #
-    # Start with an overview
-    # World
-    bbox = (-180.0,-90.0, 180.0,90.0)
-
-    render_tiles(bbox, mapfile, tile_dir, 0, 5, "World")
-
-    minZoom = 10
-    maxZoom = 16
-    bbox = (-2, 50.0,1.0,52.0)
-    render_tiles(bbox, mapfile, tile_dir, minZoom, maxZoom)
-
-    # Muenchen
-    bbox = (11.4,48.07, 11.7,48.22)
-    render_tiles(bbox, mapfile, tile_dir, 1, 12 , "Muenchen")
-
-    # Muenchen+
-    bbox = (11.3,48.01, 12.15,48.44)
-    render_tiles(bbox, mapfile, tile_dir, 7, 12 , "Muenchen+")
-
-    # Muenchen++
-    bbox = (10.92,47.7, 12.24,48.61)
-    render_tiles(bbox, mapfile, tile_dir, 7, 12 , "Muenchen++")
-
-    # Nuernberg
-    bbox=(10.903198,49.560441,49.633534,11.038085)
-    render_tiles(bbox, mapfile, tile_dir, 10, 16, "Nuernberg")
-
-    # Karlsruhe
-    bbox=(8.179113,48.933617,8.489252,49.081707)
-    render_tiles(bbox, mapfile, tile_dir, 10, 16, "Karlsruhe")
-
-    # Karlsruhe+
-    bbox = (8.3,48.95,8.5,49.05)
-    render_tiles(bbox, mapfile, tile_dir, 1, 16, "Karlsruhe+")
-
-    # Augsburg
-    bbox = (8.3,48.95,8.5,49.05)
-    render_tiles(bbox, mapfile, tile_dir, 1, 16, "Augsburg")
-
-    # Augsburg+
-    bbox=(10.773251,48.369594,10.883834,48.438577)
-    render_tiles(bbox, mapfile, tile_dir, 10, 14, "Augsburg+")
-
-    # Europe+
-    bbox = (1.0,10.0, 20.6,50.0)
-    render_tiles(bbox, mapfile, tile_dir, 1, 11 , "Europe+")
+        bbox = [ float (x) for x in sys.argv[2:6] ]
+    except ValueError:
+        # treat each argv as a triple deg,min,sec
+        bbox= [ int(d) + float(m)/60 + float(s)/3600 for d, m, s in [ arg.split (',') for arg in sys.argv[2:6] ] ]
+
+    print bbox
+    render_tiles(bbox, mapfile, tile_dir, int (sys.argv[6]), int (sys.argv[7]), "foo")
