diff --git a/inc/entities.xml.inc b/inc/entities.xml.inc
index 66d50f6..0f56c99 100644
--- a/inc/entities.xml.inc
+++ b/inc/entities.xml.inc
@@ -44,3 +44,5 @@
 <!ENTITY minscale_zoom17 "<MinScaleDenominator>2500</MinScaleDenominator>">
 <!ENTITY maxscale_zoom18 "<MaxScaleDenominator>2500</MaxScaleDenominator>">
 <!ENTITY minscale_zoom18 "">
+<!ENTITY % colors SYSTEM "colors.xml.inc">
+%colors;
diff --git a/inc/fontset-settings.xml.inc.template b/inc/fontset-settings.xml.inc.template
index 9b83544..73854a5 100644
--- a/inc/fontset-settings.xml.inc.template
+++ b/inc/fontset-settings.xml.inc.template
@@ -4,13 +4,13 @@ ls `python -c "import mapnik;print mapnik.fontscollectionpath"`-->

 <FontSet name="book-fonts">
   <Font face-name="DejaVu Sans Book" />
-  <Font face-name="unifont Medium" />
+  <!--Font face-name="unifont Medium" /-->
 </FontSet>
 <FontSet name="bold-fonts">
   <Font face-name="DejaVu Sans Bold" />
-  <Font face-name="unifont Medium" />
+  <!--Font face-name="unifont Medium" /-->
 </FontSet>
 <FontSet name="oblique-fonts">
   <Font face-name="DejaVu Sans Oblique" />
-  <Font face-name="unifont Medium" />
+  <!--Font face-name="unifont Medium" /-->
 </FontSet>
diff --git a/inc/layer-addressing.xml.inc b/inc/layer-addressing.xml.inc
index 7c8ee76..4e61d22 100644
--- a/inc/layer-addressing.xml.inc
+++ b/inc/layer-addressing.xml.inc
@@ -1,9 +1,9 @@
-<Style name="interpolation">
+<!--Style name="interpolation">
     <Rule>
       &maxscale_zoom17;
       <LineSymbolizer stroke="#888" stroke-width="1" stroke-dasharray="2,4"/>
     </Rule>
-</Style>
+</Style-->
 <Style name="housenumbers">
     <Rule>
       &maxscale_zoom17;
@@ -21,7 +21,7 @@
       <TextSymbolizer size="9" fill="#444" dy="0" fontset-name="book-fonts" wrap-width="20" placement="interior">[addr:housename]</TextSymbolizer>
     </Rule>
 </Style>
-<Layer name="interpolation_lines" status="on" srs="&osm2pgsql_projection;">
+<!--Layer name="interpolation_lines" status="on" srs="&osm2pgsql_projection;">
      <StyleName>interpolation</StyleName>
      <Datasource>
       <Parameter name="table">
@@ -29,7 +29,7 @@
       </Parameter>
       &datasource-settings;
     </Datasource>
-</Layer>
+</Layer-->
 <Layer name="housenumbers" status="on" srs="&osm2pgsql_projection;">
     <StyleName>housenumbers</StyleName>
     <Datasource>
diff --git a/inc/layer-admin.xml.inc b/inc/layer-admin.xml.inc
index e9a522e..a130bf7 100644
--- a/inc/layer-admin.xml.inc
+++ b/inc/layer-admin.xml.inc
@@ -1,4 +1,4 @@
-<Style name="admin-01234">
+<!--Style name="admin-01234">
     <Rule>
       <Filter>[admin_level]='2' or [admin_level]='3'</Filter>
       &maxscale_zoom4;
@@ -38,8 +38,8 @@
       &maxscale_zoom11;
       <LineSymbolizer stroke="purple" stroke-width="3" stroke-dasharray="4,3" stroke-opacity="0.2"/>
     </Rule>
-</Style>
-<Style name="admin-5678">
+</Style-->
+<!--Style name="admin-5678">
     <Rule>
       <Filter>[admin_level]='5'</Filter>
       &maxscale_zoom11;
@@ -55,8 +55,8 @@
       &maxscale_zoom12;
       <LineSymbolizer stroke="purple" stroke-width="1.5" stroke-dasharray="5,2" stroke-opacity="0.3"/>
     </Rule>
-</Style>
-<Style name="admin-other">
+</Style-->
+<!--Style name="admin-other">
     <Rule>
       <Filter>[admin_level]='10' or [admin_level]='9'</Filter>
       &maxscale_zoom13;
@@ -68,9 +68,9 @@
       &minscale_zoom11;
       <LineSymbolizer stroke="purple" stroke-width="1" stroke-opacity="0.2"/>
     </Rule>
-</Style>
+</Style-->

-<Layer name="admin-01234" status="on" srs="&osm2pgsql_projection;">
+<!--Layer name="admin-01234" status="on" srs="&osm2pgsql_projection;">
     <StyleName>admin-01234</StyleName>
     <Datasource>
       <Parameter name="table">
@@ -105,4 +105,4 @@
        ) as admin</Parameter>
       &datasource-settings;
     </Datasource>
-</Layer>
+</Layer-->
diff --git a/inc/layer-amenity-points.xml.inc b/inc/layer-amenity-points.xml.inc
index 25893ca..e8e0f4a 100644
--- a/inc/layer-amenity-points.xml.inc
+++ b/inc/layer-amenity-points.xml.inc
@@ -56,7 +56,7 @@
       <PointSymbolizer file="&symbols;/cafe.p.16.png" placement="interior"/>
     </Rule>
     <Rule>
-      &maxscale_zoom16;
+      &maxscale_zoom13;
       <Filter>[tourism]='camp_site'</Filter>
       <PointSymbolizer file="&symbols;/camping.n.16.png" placement="interior"/>
     </Rule>
@@ -66,7 +66,7 @@
       <PointSymbolizer file="&symbols;/transport_ford.p.16.png" placement="interior"/>
     </Rule>
     <Rule>
-      &maxscale_zoom16;
+      &maxscale_zoom13;
       <Filter>[tourism]='caravan_site'</Filter>
       <PointSymbolizer file="&symbols;/caravan_park.p.24.png" placement="interior"/>
     </Rule>
@@ -91,7 +91,7 @@
       <PointSymbolizer file="&symbols;/firestation.p.16.png" placement="interior"/>
     </Rule>
     <Rule>
-      &maxscale_zoom17;
+      &maxscale_zoom13;
       <Filter>[amenity]='fuel'</Filter>
       <PointSymbolizer file="&symbols;/fuel.p.16.png" placement="interior"/>
     </Rule>
@@ -106,7 +106,7 @@
       <PointSymbolizer file="&symbols;/bandb.p.20.png" placement="interior"/>
     </Rule>
     <Rule>
-      &maxscale_zoom15;
+      &maxscale_zoom13;
       <Filter>[amenity]='hospital'</Filter>
       <PointSymbolizer file="&symbols;/hospital.p.16.png" placement="interior"/>
     </Rule>
@@ -126,7 +126,7 @@
       <PointSymbolizer file="&symbols;/motel.p.20.png" placement="interior"/>
     </Rule>
     <Rule>
-      &maxscale_zoom16;
+      &maxscale_zoom14;
       <Filter>[tourism]='information'</Filter>
       <PointSymbolizer file="&symbols;/information.p.16.png" placement="interior"/>
     </Rule>
@@ -161,7 +161,7 @@
       <PointSymbolizer file="&symbols;/museum.p.16.png" placement="interior"/>
     </Rule>
     <Rule>
-      &maxscale_zoom15;
+      &maxscale_zoom14;
       <Filter>[amenity] = 'parking' and ([access] = 'public' or [access] = 'yes' or not [access] != '')</Filter>
       <PointSymbolizer file="&symbols;/parking.p.16.png" allow-overlap="false" placement="interior"/>
     </Rule>
@@ -171,7 +171,7 @@
       <PointSymbolizer file="&symbols;/parking_private.p.16.png" allow-overlap="false" placement="interior"/>
     </Rule>
     <Rule>
-      &maxscale_zoom17;
+      &maxscale_zoom15;
       <Filter>[amenity]='pharmacy'</Filter>
       <PointSymbolizer file="&symbols;/pharmacy.p.16.png" placement="interior"/>
     </Rule>
@@ -201,7 +201,7 @@
       <PointSymbolizer file="&symbols;/place_of_worship3.p.16.png" allow-overlap="false" placement="interior"/>
      </Rule>
      <Rule>
-      &maxscale_zoom17;
+      &maxscale_zoom14;
       <Filter>[amenity]='police'</Filter>
       <PointSymbolizer file="&symbols;/police.p.16.png" placement="interior"/>
     </Rule>
@@ -256,7 +256,7 @@
       <PointSymbolizer file="&symbols;/theatre.p.20.png" placement="interior"/>
     </Rule>
     <Rule>
-      &maxscale_zoom17;
+      &maxscale_zoom15;
       <Filter>[amenity]='toilets'</Filter>
       <PointSymbolizer file="&symbols;/toilets.p.20.png" placement="interior"/>
     </Rule>
@@ -271,7 +271,7 @@
       <PointSymbolizer file="&symbols;/amenity_prison.p.20.png" placement="interior"/>
     </Rule>
     <Rule>
-      &maxscale_zoom16;
+      &maxscale_zoom13;
       <Filter>[tourism]='viewpoint'</Filter>
       <PointSymbolizer file="&symbols;/view_point.p.16.png" placement="interior"/>
     </Rule>
@@ -296,12 +296,12 @@
       <PointSymbolizer file="&symbols;/tower_water.p.20.png" placement="interior"/>
     </Rule>
     <Rule>
-      &maxscale_zoom17;
+      &maxscale_zoom13;
       <Filter>[historic]='memorial'</Filter>
       <PointSymbolizer file="&symbols;/tourist_memorial.p.20.png" placement="interior"/>
     </Rule>
     <Rule>
-      &maxscale_zoom16;
+      &maxscale_zoom13;
       <Filter>[historic]='archaeological_site'</Filter>
       <PointSymbolizer file="&symbols;/tourist_archaeological2.glow.16.png" placement="interior"/>
     </Rule>
diff --git a/inc/layer-amenity-symbols.xml.inc b/inc/layer-amenity-symbols.xml.inc
index 802aaa1..d37e527 100644
--- a/inc/layer-amenity-symbols.xml.inc
+++ b/inc/layer-amenity-symbols.xml.inc
@@ -67,7 +67,7 @@
       <Filter>[natural]='spring'</Filter>
       <PointSymbolizer file="&symbols;/spring.png" placement="interior"/>
     </Rule>
-    <Rule>
+    <!--Rule>
       &maxscale_zoom16;
       &minscale_zoom16;
       <Filter>[natural]='tree'</Filter>
@@ -77,7 +77,7 @@
       &maxscale_zoom17;
       <Filter>[natural]='tree'</Filter>
       <PointSymbolizer file="&symbols;/tree2.png" ignore-placement="true" placement="interior"/>
-    </Rule>
+    </Rule-->
     <Rule>
       &maxscale_zoom15;
       <Filter>([power]='generator' and ([generator:source] = 'wind' or [power_source] = 'wind')) or [man_made]='power_wind'</Filter>
diff --git a/inc/layer-buildings.xml.inc b/inc/layer-buildings.xml.inc
index 2377a19..141c2a8 100644
--- a/inc/layer-buildings.xml.inc
+++ b/inc/layer-buildings.xml.inc
@@ -23,7 +23,7 @@
     </Rule>
 </Style>

-<Style name="buildings">
+<!--Style name="buildings">
     <Rule>
       <Filter>[building] = 'INT-light'</Filter>
       &maxscale_zoom12;
@@ -49,7 +49,7 @@
       &maxscale_zoom14;
       <LineSymbolizer stroke="#330066" stroke-width="0.2"/>
     </Rule>
-</Style>
+</Style-->

 <!-- Render only select building types starting at z10. -->
 <Layer name="buildings-lz" status="on" srs="&osm2pgsql_projection;">
diff --git a/inc/layer-landcover.xml.inc b/inc/layer-landcover.xml.inc
index 8dd35e7..2a1042f 100644
--- a/inc/layer-landcover.xml.inc
+++ b/inc/layer-landcover.xml.inc
@@ -1,10 +1,10 @@
 <Style name="landcover">
-     <Rule>
+     <!--Rule>
       <Filter>[leisure] = 'swimming_pool'</Filter>
       &maxscale_zoom14;
       <PolygonSymbolizer fill="#b5d0d0"/>
       <LineSymbolizer stroke="blue" stroke-width="0.5"/>
-    </Rule>
+    </Rule-->
     <Rule>
       <Filter>[leisure] = 'playground'</Filter>
       &maxscale_zoom13;
@@ -22,65 +22,65 @@
       &maxscale_zoom10;
       <PolygonSymbolizer fill="#f2caea"/>
     </Rule>
-    <Rule>
+    <!--Rule>
       <Filter>[landuse] = 'quarry'</Filter>
       &maxscale_zoom11;
       <PolygonPatternSymbolizer file="&symbols;/quarry2.png" />
       <LineSymbolizer stroke="grey" stroke-width="0.5"/>
-    </Rule>
-    <Rule>
+    </Rule-->
+    <!--Rule>
       <Filter>[landuse] = 'vineyard'</Filter>
       &maxscale_zoom10;
       &minscale_zoom12;
       <PolygonSymbolizer fill="#abdf96"/>
-    </Rule>
+    </Rule-->
     <Rule>
       <Filter>[landuse] = 'orchard'</Filter>
       &maxscale_zoom10;
-      <PolygonPatternSymbolizer file="&symbols;/orchard.png" />
+      <PolygonPatternSymbolizer file="&symbols;/orchard_trans.png" />
     </Rule>
-    <Rule>
+    <!--Rule>
       <Filter>[landuse] = 'vineyard'</Filter>
       &maxscale_zoom13;
       <PolygonPatternSymbolizer file="&symbols;/vineyard.png" />
-    </Rule>
+    </Rule-->
     <Rule>
       &maxscale_zoom10;
       &minscale_zoom14;
       <Filter>[landuse] = 'cemetery' or [landuse] = 'grave_yard' or [amenity]='grave_yard'</Filter>
       <PolygonSymbolizer fill="#aacbaf"/>
     </Rule>
-    <Rule>
+    <!--Rule>
       &maxscale_zoom10;
       &minscale_zoom18;
       <Filter>[landuse] = 'residential'</Filter>
       <PolygonSymbolizer fill="#ddd"/>
-    </Rule>
-    <Rule>
+    </Rule-->
+    <!--Rule>
       &maxscale_zoom12;
       &minscale_zoom18;
       <Filter>[landuse] = 'garages'</Filter>
       <PolygonSymbolizer fill-opacity="0.2" fill="#996"/>
-    </Rule>
-    <Rule>
+    </Rule-->
+    <!--Rule>
       &maxscale_zoom10;
       &minscale_zoom18;
       <Filter>[military] = 'barracks'</Filter>
       <PolygonSymbolizer fill="#ff8f8f"/>
-    </Rule>
-    <Rule>
+    </Rule-->
+    <!--Rule>
       &maxscale_zoom10;
       &minscale_zoom13;
       <Filter>[landuse]='field' or [natural]='field'</Filter>
       <PolygonSymbolizer fill-opacity="0.2" fill="#666600"/>
-    </Rule>
-    <Rule>
+    </Rule-->
+    <!--Rule>
       &maxscale_zoom14;
       &minscale_zoom18;
       <Filter>[landuse]='field' or [natural]='field'</Filter>
       <PolygonSymbolizer fill-opacity="0.2" fill="#666600"/>
       <LineSymbolizer stroke="#660" stroke-width="0.3" stroke-opacity="0.4"/>
-    </Rule>
+    </Rule-->
     <Rule>
       &maxscale_zoom9;
       &minscale_zoom10;
@@ -108,11 +108,11 @@
       <Filter>([landuse] = 'cemetery' or [landuse] = 'grave_yard' or [amenity]='grave_yard') and [religion] = 'INT-generic'</Filter>
       <PolygonPatternSymbolizer file="&symbols;/grave_yard_generic.png" />
     </Rule>
-    <Rule>
+    <!--Rule>
       &maxscale_zoom10;
       <Filter>[landuse] = 'meadow' or [landuse] = 'grass'</Filter>
       <PolygonSymbolizer fill="#cfeca8"/>
-    </Rule>
+    </Rule-->
     <Rule>
       <Filter>[leisure] = 'park' or [leisure] = 'recreation_ground'</Filter>
       &maxscale_zoom10;
@@ -123,11 +123,11 @@
       &maxscale_zoom10;
       <PolygonPatternSymbolizer file="&symbols;/zoo.png" />
     </Rule>
-    <Rule>
+    <!--Rule>
       <Filter>[leisure] = 'common'</Filter>
       &maxscale_zoom10;
       <PolygonSymbolizer fill="#cfeca8"/>
-    </Rule>
+    </Rule-->
     <Rule>
       <Filter>[leisure] = 'garden'</Filter>
       &maxscale_zoom10;
@@ -138,7 +138,7 @@
       &maxscale_zoom10;
       <PolygonSymbolizer fill="#b5e3b5"/>
     </Rule>
-    <Rule>
+    <!--Rule>
       <Filter>[landuse] = 'allotments'</Filter>
       &maxscale_zoom10;
       &minscale_zoom13;
@@ -148,19 +148,19 @@
       <Filter>[landuse] = 'allotments'</Filter>
       &maxscale_zoom14;
       <PolygonPatternSymbolizer file="&symbols;/allotments.png" />
-    </Rule>
-    <Rule>
+    </Rule-->
+    <!--Rule>
       <Filter>[landuse] = 'forest'</Filter>
       &maxscale_zoom8;
       &minscale_zoom13;
       <PolygonSymbolizer fill="#8dc56c"/>
-    </Rule>
+    </Rule-->
     <Rule>
       <Filter>[landuse] = 'forest'</Filter>
       &maxscale_zoom14;
-      <PolygonPatternSymbolizer file="&symbols;/forest.png" />
+      <PolygonPatternSymbolizer file="&symbols;/forest_trans.png" />
     </Rule>
-    <Rule>
+    <!--Rule>
       <Filter>[landuse]='farmyard'</Filter>
       &maxscale_zoom9;
       <PolygonSymbolizer fill="#ddbf92"/>
@@ -169,7 +169,7 @@
       <Filter>[landuse] = 'farm' or [landuse]='farmland'</Filter>
       &maxscale_zoom9;
       <PolygonSymbolizer fill="#ead8bd"/>
-    </Rule>
+    </Rule-->
     <Rule>
       <Filter>[landuse] = 'recreation_ground' or [landuse] = 'conservation'</Filter>
       &maxscale_zoom10;
@@ -180,7 +180,7 @@
       &maxscale_zoom11;
       <PolygonSymbolizer fill="#cfeca8"/>
     </Rule>
-    <Rule>
+    <!--Rule>
       <Filter>[landuse] = 'retail'</Filter>
       &maxscale_zoom10;
       <PolygonSymbolizer fill="#f1dada"/>
@@ -189,13 +189,13 @@
       <Filter>[landuse] = 'retail'</Filter>
       &maxscale_zoom15;
       <LineSymbolizer stroke="red" stroke-width="0.3"/>
-    </Rule>
-    <Rule>
+    </Rule-->
+    <!--Rule>
       <Filter>[landuse] = 'industrial' or [landuse]='railway'</Filter>
       &maxscale_zoom10;
       <PolygonSymbolizer fill="#dfd1d6"/>
-    </Rule>
-    <Rule>
+    </Rule-->
+    <!--Rule>
       <Filter>[power] = 'station' or [power]='generator'</Filter>
       &maxscale_zoom10;
       &minscale_zoom11;
@@ -212,18 +212,18 @@
       &maxscale_zoom13;
       <PolygonSymbolizer fill="#bbb"/>
       <LineSymbolizer stroke="#555" stroke-width="0.4"/>
-    </Rule>
-    <Rule>
+    </Rule-->
+    <!--Rule>
       <Filter>[landuse] = 'commercial'</Filter>
       &maxscale_zoom10;
       <PolygonSymbolizer fill="#efc8c8"/>
-    </Rule>
-    <Rule>
+    </Rule-->
+    <!--Rule>
       <Filter>[landuse] = 'brownfield' or [landuse]='landfill' or [landuse] = 'greenfield' or [landuse]='construction'</Filter>
       &maxscale_zoom10;
       <PolygonSymbolizer fill-opacity="0.7" fill="#9d9d6c"/>
-    </Rule>
-     <Rule>
+    </Rule-->
+     <!--Rule>
       <Filter>[natural] = 'wood' or [landuse] = 'wood'</Filter>
       &maxscale_zoom8;
       <PolygonSymbolizer fill="#aed1a0"/>
@@ -253,11 +253,11 @@
       &maxscale_zoom10;
       &minscale_zoom13;
       <PolygonSymbolizer fill="#b5e3b5"/>
-    </Rule>
+    </Rule-->
     <Rule>
       &maxscale_zoom14;
       <Filter>[natural] = 'scrub'</Filter>
-      <PolygonPatternSymbolizer file="&symbols;/scrub.png" />
+      <PolygonPatternSymbolizer file="&symbols;/scrub_trans.png" />
     </Rule>
     <Rule>
       <Filter>[amenity] = 'university' or [amenity] = 'college' or [amenity] = 'school' or [amenity]='hospital' or [amenity]='kindergarten'</Filter>
@@ -285,15 +285,15 @@
       <PolygonPatternSymbolizer file="&symbols;/amenity_prison.png" />
       <LineSymbolizer stroke="#555" stroke-width="0.3"/>
     </Rule>
-    <Rule>
+    <!--Rule>
       <Filter>[aeroway] = 'apron'</Filter>
       &maxscale_zoom12;
       <PolygonSymbolizer fill="#e9d1ff"/>
-    </Rule>
+    </Rule-->
     <Rule>
       <Filter>[aeroway] = 'aerodrome'</Filter>
       &maxscale_zoom12;
-      <PolygonSymbolizer fill-opacity="0.2" fill="#ccc"/>
+      <!--PolygonSymbolizer fill-opacity="0.2" fill="#ccc"/-->
       <LineSymbolizer stroke="#555" stroke-width="0.2"/>
     </Rule>
     <Rule>
diff --git a/inc/layer-placenames.xml.inc b/inc/layer-placenames.xml.inc
index afbd3ee..956c486 100644
--- a/inc/layer-placenames.xml.inc
+++ b/inc/layer-placenames.xml.inc
@@ -190,4 +190,3 @@
       &datasource-settings;
     </Datasource>
 </Layer>
-
diff --git a/inc/layer-power.xml.inc b/inc/layer-power.xml.inc
index 77ad41a..a3a43b0 100644
--- a/inc/layer-power.xml.inc
+++ b/inc/layer-power.xml.inc
@@ -1,4 +1,4 @@
-<Style name="power_line">
+<!--Style name="power_line">
     <Rule>
       &maxscale_zoom14;
       &minscale_zoom15;
@@ -64,5 +64,5 @@
       <Parameter name="table">(select way from &prefix;_point where power='pole') as power_poles</Parameter>
       &datasource-settings;
     </Datasource>
-</Layer>
+</Layer-->

diff --git a/inc/layer-shapefiles.xml.inc b/inc/layer-shapefiles.xml.inc
index 14da607..d243443 100644
--- a/inc/layer-shapefiles.xml.inc
+++ b/inc/layer-shapefiles.xml.inc
@@ -13,7 +13,7 @@
      <TextSymbolizer size="8" dy="2" fill="grey" fontset-name="book-fonts" halo-radius="1">[NAMEASCII]</TextSymbolizer>
     </Rule>
 </Style>
-<Style name="world">
+<!--Style name="world">
     <Rule>
       &maxscale_zoom0;
       &minscale_zoom9;
@@ -25,16 +25,16 @@
       &maxscale_zoom10;
       <PolygonSymbolizer fill="#f2efe9"/>
     </Rule>
-</Style>
+</Style-->
   <!-- BUILTUP VMAP0 -->
-<Style name="builtup">
+<!--Style name="builtup">
     <Rule>
       &maxscale_zoom8;
       &minscale_zoom10;
       <PolygonSymbolizer fill="#ddd"/>
     </Rule>
-</Style>
-<Layer name="world" status="on" srs="&srs900913;">
+</Style-->
+<!--Layer name="world" status="on" srs="&srs900913;">
     <StyleName>world</StyleName>
     <Datasource>
       <Parameter name="type">shape</Parameter>
@@ -54,7 +54,7 @@
       <Parameter name="type">shape</Parameter>
       <Parameter name="file">&world_boundaries;/builtup_area</Parameter>
     </Datasource>
-</Layer>
+</Layer-->
 <Layer name="necountries" status="on" srs="&srs4326;">
     <StyleName>necountries</StyleName>
     <Datasource>
diff --git a/inc/layer-water.xml.inc b/inc/layer-water.xml.inc
index f819bc4..f59e2f5 100644
--- a/inc/layer-water.xml.inc
+++ b/inc/layer-water.xml.inc
@@ -32,11 +32,11 @@
       &maxscale_zoom13;
       <PolygonPatternSymbolizer file="&symbols;/mud.png" />
     </Rule>
-    <Rule>
+    <!--Rule>
       <Filter>[natural] = 'land'</Filter>
       &maxscale_zoom10;
       <PolygonSymbolizer fill="#f2efe9"/>
-    </Rule>
+    </Rule-->
 </Style>
 <Style name="water-areas-overlay">
     <Rule>
diff --git a/inc/layers.xml.inc b/inc/layers.xml.inc
index 2405e5b..08e2e14 100644
--- a/inc/layers.xml.inc
+++ b/inc/layers.xml.inc
@@ -13,3 +13,5 @@
 <!ENTITY layer-power SYSTEM "layer-power.xml.inc">
 <!ENTITY layer-addressing SYSTEM "layer-addressing.xml.inc">
 <!ENTITY layer-admin SYSTEM "layer-admin.xml.inc">
+<!ENTITY layer-terrain SYSTEM "layer-terrain.xml.inc">
+<!ENTITY layer-contour SYSTEM "layer-contour.xml.inc">
diff --git a/osm.xml b/osm.xml
index 0dd195b..ba43479 100644
--- a/osm.xml
+++ b/osm.xml
@@ -148,22 +148,22 @@
       <Filter>[highway] = 'living_street'</Filter>
       &maxscale_zoom14;
       <PolygonSymbolizer fill="#ccc"/>
-   </Rule>
-   <Rule>
+    </Rule>
+    <Rule>
       <Filter>[highway] = 'residential' or [highway]='unclassified' or [highway]='service'</Filter>
       &maxscale_zoom14;
       <PolygonSymbolizer fill="#fff"/>
-   </Rule>
-   <Rule>
+    </Rule>
+    <Rule>
       <Filter>[highway] = 'pedestrian' or [highway]='footway' or [highway]='path'</Filter>
       &maxscale_zoom14;
       <PolygonSymbolizer fill="#ededed"/>
-   </Rule>
-   <Rule>
+    </Rule>
+    <!--Rule>
       <Filter>[highway] = 'track'</Filter>
       &maxscale_zoom14;
       <PolygonSymbolizer fill="#dfcc66"/>
-   </Rule>
+    </Rule-->
     <Rule>
       <Filter>[highway] = 'platform' or [railway] = 'platform'</Filter>
       &maxscale_zoom16;
@@ -174,11 +174,11 @@
       &maxscale_zoom11;
       <PolygonSymbolizer fill="#bbc"/>
     </Rule>
-    <Rule>
+    <!--Rule>
       <Filter>[aeroway] = 'taxiway'</Filter>
       &maxscale_zoom13;
       <PolygonSymbolizer fill="#bbc"/>
-    </Rule>
+    </Rule-->
     <Rule>
       <Filter>[aeroway] = 'helipad'</Filter>
       &maxscale_zoom16;
@@ -543,6 +543,7 @@
       &minscale_zoom18;
       <LineSymbolizer stroke="#506077" stroke-width="13" stroke-dasharray="4,2"/>
     </Rule>
+
     <Rule>
       <Filter>[highway] = 'trunk' or [highway] = 'trunk_link'</Filter>
       &maxscale_zoom12;
@@ -567,6 +568,7 @@
       &minscale_zoom18;
       <LineSymbolizer stroke="#477147" stroke-width="14" stroke-dasharray="4,2"/>
     </Rule>
+
     <Rule>
       <Filter>[highway] = 'primary' or [highway] = 'primary_link'</Filter>
       &maxscale_zoom12;
@@ -591,6 +593,7 @@
       &minscale_zoom18;
       <LineSymbolizer stroke="#8d4346" stroke-width="14" stroke-dasharray="4,2"/>
     </Rule>
+
     <Rule>
       <Filter>[highway] = 'secondary' or [highway] = 'secondary_link'</Filter>
       &maxscale_zoom12;
@@ -615,6 +618,7 @@
       &minscale_zoom18;
       <LineSymbolizer stroke="#a37b48" stroke-width="17" stroke-dasharray="4,2"/>
     </Rule>
+
     <Rule>
       <Filter>[highway] = 'tertiary' or [highway]='tertiary_link'</Filter>
       &maxscale_zoom13;
@@ -627,6 +631,7 @@
       &minscale_zoom14;
       <LineSymbolizer stroke="#999" stroke-width="7.5" stroke-dasharray="4,2"/>
     </Rule>
+
     <Rule>
       <Filter>[highway] = 'residential' or [highway] = 'unclassified' or [highway] = 'road'</Filter>
       &maxscale_zoom13;
@@ -651,6 +656,7 @@
       &minscale_zoom16;
       <LineSymbolizer stroke="#999" stroke-width="11" stroke-dasharray="4,2"/>
     </Rule>
+
     <Rule>
       <Filter>[highway] = 'tertiary' or [highway]='tertiary_link'</Filter>
       &maxscale_zoom15;
@@ -664,6 +670,7 @@
       <LineSymbolizer stroke="#999" stroke-width="16" stroke-dasharray="4,2"/>
     </Rule>
 </Style>
+
 <Style name="tunnels-fill">
     <Rule>
       <Filter>[highway] = 'motorway' or [highway]='motorway_link'</Filter>
@@ -816,6 +823,7 @@
       <LineSymbolizer stroke-linejoin="round" stroke="#fff" stroke-width="13" stroke-linecap="round"/>
     </Rule>
 </Style>
+
 <Style name="minor-roads-casing-links">
     <Rule>
       <Filter>[highway]='raceway'</Filter>
@@ -835,6 +843,7 @@
       &minscale_zoom18;
       <LineSymbolizer stroke-linejoin="round" stroke="pink" stroke-width="7"/>
     </Rule>
+
     <Rule>
       <Filter>[highway]='motorway_link' and not [tunnel] = 'yes'</Filter>
       &maxscale_zoom12;
@@ -1331,7 +1340,7 @@
     </Rule>
 </Style>
 <Style name="minor-roads-fill">
-    <Rule>
+    <!--Rule>
       &maxscale_zoom12;
       &minscale_zoom12;
       <Filter>([highway] = 'proposed' or [highway]='construction') and not [construction] != ''</Filter>
@@ -1484,7 +1493,7 @@
       <Filter>([highway] = 'proposed' or [highway]='construction') and [construction]='cycleway'</Filter>
       <LineSymbolizer stroke-linejoin="round" stroke="white" stroke-width="3" stroke-linecap="round" stroke-opacity="0.4"/>
       <LineSymbolizer stroke-linejoin="round" stroke="#69f" stroke-width="1.2" stroke-dasharray="2,6" stroke-linecap="round"/>
-    </Rule>
+    </Rule-->
     <Rule>
       <Filter>[highway] = 'motorway' and not [tunnel] = 'yes'</Filter>
       &maxscale_zoom12;
@@ -1593,12 +1602,12 @@
       <LineSymbolizer stroke-linejoin="round" stroke="#a1a1a1" stroke-width="3" stroke-dasharray="0,5,1,4"/>
       <LineSymbolizer stroke-linejoin="round" stroke="#9b9b9b" stroke-width="3" stroke-dasharray="0,6,1,3"/>
     </Rule>
-    <Rule>
+    <!--Rule>
       &maxscale_zoom13;
       &minscale_zoom18;
       <Filter>([railway] = 'disused' or [railway] = 'abandoned' or [railway]='construction') and not [highway] != ''</Filter>
       <LineSymbolizer stroke-linejoin="round" stroke="grey" stroke-width="2" stroke-dasharray="2,4"/>
-    </Rule>
+    </Rule-->
     <Rule>
       &maxscale_zoom13;
       &minscale_zoom13;
@@ -1609,7 +1618,7 @@
     <Rule>
       &maxscale_zoom14;
       &minscale_zoom18;
-      <Filter>[railway] = 'rail' and not [tunnel]='yes'</Filter>
+      <Filter>[railway] = 'rail' and not [tunnel] = 'yes'</Filter>
       <LineSymbolizer stroke-linejoin="round" stroke="#999999" stroke-width="3"/>
       <LineSymbolizer stroke-linejoin="round" stroke="white" stroke-width="1" stroke-dasharray="0,11,8,1"/>
     </Rule>
@@ -1913,12 +1922,12 @@
       <LineSymbolizer stroke-linejoin="round" stroke="white" stroke-width="3" stroke-linecap="round" stroke-opacity="0.4"/>
       <LineSymbolizer stroke-linejoin="round" stroke="blue" stroke-width="1.2" stroke-dasharray="1,3" stroke-linecap="round"/>
     </Rule>
-    <Rule>
+    <!--Rule>
       <Filter>[highway] = 'byway'</Filter>
       &maxscale_zoom13;
       <LineSymbolizer stroke-linejoin="round" stroke="white" stroke-width="4" stroke-linecap="round" stroke-opacity="0.4"/>
       <LineSymbolizer stroke-linejoin="round" stroke="#ffcc00" stroke-width="1.5" stroke-dasharray="3,4" stroke-linecap="round"/>
-    </Rule>
+    </Rule-->
     <Rule>
       <Filter>[highway] = 'track'</Filter>
       &maxscale_zoom13;
@@ -3152,7 +3161,7 @@
       <Filter>[highway] = 'trunk' or [highway] = 'primary'</Filter>
       &maxscale_zoom13;
       &minscale_zoom13;
-      <TextSymbolizer size="8" fill="black" placement="line" fontset-name="book-fonts" halo-radius="0">[name]</TextSymbolizer>
+      <TextSymbolizer size="8" fill="black" placement="line" fontset-name="book-fonts" halo-radius="1">[name]</TextSymbolizer>
     </Rule>
     <Rule>
       <Filter>[highway] = 'secondary'</Filter>
@@ -3164,13 +3173,13 @@
       <Filter>[highway] = 'trunk' or [highway] = 'primary'</Filter>
       &maxscale_zoom14;
       &minscale_zoom14;
-      <TextSymbolizer size="9" fill="black" placement="line" fontset-name="book-fonts" halo-radius="0">[name]</TextSymbolizer>
+      <TextSymbolizer size="9" fill="black" placement="line" fontset-name="book-fonts" halo-radius="1">[name]</TextSymbolizer>
     </Rule>
     <Rule>
       <Filter>[highway] = 'trunk' or [highway] = 'primary'</Filter>
       &maxscale_zoom15;
       &minscale_zoom18;
-      <TextSymbolizer size="10" fill="black" placement="line" fontset-name="book-fonts" halo-radius="0">[name]</TextSymbolizer>
+      <TextSymbolizer size="10" fill="black" placement="line" fontset-name="book-fonts" halo-radius="1">[name]</TextSymbolizer>
     </Rule>
     <Rule>
       <Filter>[highway] = 'secondary'</Filter>
@@ -3195,7 +3204,7 @@
       &minscale_zoom18;
       <TextSymbolizer size="11" fill="#000" placement="line" fontset-name="book-fonts" halo-radius="1">[name]</TextSymbolizer>
     </Rule>
-    <Rule>
+    <!--Rule>
       &maxscale_zoom13;
       &minscale_zoom15;
       <Filter>[highway] = 'proposed' or [highway]='construction'</Filter>
@@ -3206,7 +3215,7 @@
       &minscale_zoom18;
       <Filter>[highway] = 'proposed' or [highway]='construction'</Filter>
       <TextSymbolizer size="11" fill="#000" placement="line" fontset-name="book-fonts" halo-radius="1">[name]</TextSymbolizer>
-    </Rule>
+    </Rule-->
     <Rule>
       <Filter>[highway] = 'unclassified' or [highway] = 'residential'</Filter>
       &maxscale_zoom15;
@@ -3334,7 +3343,7 @@
     </Rule>
 </Style>
 <Style name="theme_park">
-<Rule>
+    <Rule>
       &maxscale_zoom13;
       &minscale_zoom14;
       <Filter>[tourism]='theme_park'</Filter>
@@ -3348,8 +3357,10 @@
     </Rule>
 </Style>

+&layer-terrain;
 &layer-shapefiles;
 &layer-landcover;
+&layer-contour;
 &layer-water;
 &layer-water_features;
 <Layer name="tunnels" status="on" srs="&osm2pgsql_projection;">
@@ -3920,12 +3931,12 @@
       &datasource-settings;
     </Datasource>
 </Layer>
-&layer-admin;
+<!--&layer-admin;-->
 &layer-placenames;
 &layer-amenity-stations;
 &layer-amenity-symbols;
 &layer-amenity-points;
-&layer-power;
+<!--&layer-power;-->
 <Layer name="roads-text-ref-low-zoom" status="on" srs="&osm2pgsql_projection;">
      <StyleName>roads-text-ref-low-zoom</StyleName>
      <Datasource>
