Index: osm-carto/style.mss
===================================================================
--- osm-carto.orig/style.mss
+++ osm-carto/style.mss
@@ -37,19 +37,35 @@ Map {
 @motorway-fill-lowzoom: #e88b00;
 @motorway-casing: darken(@motorway-fill, @darken-lighten);
 @motorway-tunnel-fill: lighten(@motorway-fill, @darken-lighten);
+@motorway-width-z12: 6;
+@motorway-width-z13: 8;
+@motorway-width-z15: 11;
+@motorway-width-z17: 15.5; // shouldnt be narrower than trunk!
 
 @trunk-fill: #bf30bf;
 @trunk-fill-alternative: #bf30bf;
 @trunk-casing: darken(@trunk-fill, @darken-lighten);
 @trunk-tunnel-fill: lighten(@trunk-fill, @darken-lighten);
+@trunk-width-z12: 5;
+@trunk-width-z13: 7;
+@trunk-width-z15: 11;
+@trunk-width-z17: 15.5;
 
 @primary-fill: #bf3030;
 @primary-casing: darken(@primary-fill, @darken-lighten);
 @primary-tunnel-fill: lighten(@primary-fill, @darken-lighten);
+@primary-width-z12: 5;
+@primary-width-z13: 7;
+@primary-width-z15: 11;
+@primary-width-z17: 15.5;
 
 @secondary-fill: #2f6f2f;
 @secondary-casing: darken(@secondary-fill, @darken-lighten);
 @secondary-tunnel-fill: lighten(@secondary-fill, @darken-lighten);
+@secondary-width-z12: 3;
+@secondary-width-z13: 7;
+@secondary-width-z15: 11;
+@secondary-width-z17: 15.5;
 
 @tertiary-fill: #8a5c00;
 @residential-fill: #ffffff;
@@ -87,6 +103,17 @@ Map {
 @tertiary-tunnel-fill: lighten(@tertiary-fill, 5%);
 @residential-tunnel-fill: lighten(@residential-fill, 10%);
 
+@track-width-z13:        1;
+@track-grade1-width-z13: 1;
+@track-grade2-width-z13: 1;
+@track-grade1-width-z15: 1;
+@track-grade2-width-z15:          1;
+
+@bridleway-width-z13:             0.5;
+@footway-width-z13:               0.75;
+@cycleway-width-z13:              0.75;
+@path-width-z13:                  0.5;
+
 // terrain-small [0-6], terrain-medium [7-8], terrain-big [9-]
 .terrain {
   raster-scaling: lanczos;
