--- a/amenity-points.mss
+++ b/amenity-points.mss
@@ -40,7 +40,7 @@
     marker-clip: false;
   }
 
-  [feature = 'amenity_atm'][zoom >= 17] {
+  [feature = 'amenity_atm'][zoom >= @useful] {
     marker-file: url('symbols/atm.svg');
     marker-fill: @amenity-brown;
     marker-placement: interior;
@@ -219,7 +219,7 @@
     marker-clip: false;
   }
 
-  [feature = 'amenity_fuel'][zoom >= 17] {
+  [feature = 'amenity_fuel'][zoom >= @useful] {
     marker-file: url('symbols/fuel.svg');
     marker-fill: @transportation-icon;
     marker-placement: interior;
@@ -231,7 +231,7 @@
     point-placement: interior;
   }
 
-  [feature = 'amenity_hospital'][zoom >= 15] {
+  [feature = 'amenity_hospital'][zoom >= @useful] {
     marker-file: url('symbols/hospital.svg');
     marker-fill: @health-color;
     marker-placement: interior;
@@ -349,7 +349,7 @@
     }
   }
 
-  [feature = 'amenity_pharmacy'][zoom >= 17] {
+  [feature = 'amenity_pharmacy'][zoom >= @useful] {
     marker-file: url('symbols/pharmacy.svg');
     marker-fill: @health-color;
     marker-placement: interior;
@@ -420,7 +420,7 @@
     marker-clip: false;
   }
 
-  [feature = 'amenity_police'][zoom >= 16] {
+  [feature = 'amenity_police'][zoom >= @useful] {
     marker-file: url('symbols/police.svg');
     marker-fill: @amenity-brown;
     marker-placement: interior;
@@ -434,7 +434,7 @@
     marker-clip: false;
   }
 
-  [feature = 'amenity_post_office'][zoom >= 17] {
+  [feature = 'amenity_post_office'][zoom >= @useful] {
     marker-file: url('symbols/post_office.svg');
     marker-fill: @amenity-brown;
     marker-placement: interior;
@@ -501,7 +501,7 @@
     marker-clip: false;
   }
 
-  [feature = 'amenity_toilets'][zoom >= 17] {
+  [feature = 'amenity_toilets'][zoom >= @useful] {
     marker-file: url('symbols/toilets.svg');
     marker-fill: @amenity-brown;
     marker-placement: interior;
@@ -522,7 +522,7 @@
     marker-clip: false;
   }
 
-  [feature = 'tourism_viewpoint'][zoom >= 16] {
+  [feature = 'tourism_viewpoint'][zoom >= @nice] {
     marker-file: url('symbols/viewpoint.svg');
     marker-placement: interior;
     marker-fill: @amenity-brown;
@@ -551,7 +551,7 @@
     marker-clip: false;
   }
 
-  [feature = 'historic_archaeological_site'][zoom >= 16] {
+  [feature = 'historic_archaeological_site'][zoom >= @nice] {
     marker-file: url('symbols/archaeological_site.svg');
     marker-fill: @amenity-brown;
     marker-placement: interior;
@@ -566,7 +566,7 @@
     marker-clip: false;
   }
 
-  [feature = 'shop_supermarket'][zoom >= 16] {
+  [feature = 'shop_supermarket'][zoom >= @useful] {
     marker-file: url('symbols/shop/supermarket.svg');
     marker-placement: interior;
     marker-clip: false;
@@ -788,7 +788,7 @@
     marker-clip: false;
   }
 
-  [feature = 'shop_car_repair'][zoom >= 17] {
+  [feature = 'shop_car_repair'][zoom >= @useful] {
     marker-file: url('symbols/shop/car_repair.svg');
     marker-placement: interior;
     marker-clip: false;
--- a/style.mss
+++ b/style.mss
@@ -266,6 +266,14 @@
 
                 "Unifont Medium", "Unifont Upper Medium";
 
+// nice stuff is things that are not so common and that I like to see from far,
+// like castles, viewpoints and such
+@nice: 11;
+// it's also the max general zoom level
+@emergency: 14;
+// things that are useful and common enough to see in the near range
+@useful: 16;
+
 @water-color: #b5d0d0;
 @water-dark: #5A5AA1;
 @water-text: #5A5AA1;
--- a/stations.mss
+++ b/stations.mss
@@ -2,7 +2,7 @@
 @station-text: darken(saturate(@station-color, 15%), 10%);
 
 .stations {
-  [railway = 'subway_entrance'][zoom >= 18] {
+  [railway = 'subway_entrance'][zoom >= @nice] {
     marker-file: url('symbols/entrance.10.svg');
     marker-placement: interior;
     marker-fill: @transportation-icon;
