--- a/project.mml
+++ b/project.mml
@@ -555,6 +555,25 @@
         ) AS marinas_area
     properties:
       minzoom: 14
+  - id: nature-reserve-boundaries
+    name: nature-reserve-boundaries
+    geometry: polygon
+    <<: *extents
+    Datasource:
+      <<: *osm2pgsql
+      table: |-
+        (SELECT
+            way,
+            name,
+            boundary,
+            way_area/NULLIF(!pixel_width!::real*!pixel_height!::real,0) AS way_pixels
+          FROM planet_osm_polygon
+          WHERE (boundary = 'national_park' OR leisure = 'nature_reserve')
+            AND building IS NULL
+            AND way_area > 0.01*!pixel_width!::real*!pixel_height!::real
+        ) AS national_park_boundaries
+    properties:
+      minzoom: 7
   - id: piers-poly
     name: piers-poly
     geometry: polygon
@@ -1527,25 +1546,6 @@
         ) AS admin_high_zoom
     properties:
       minzoom: 13
-  - id: nature-reserve-boundaries
-    name: nature-reserve-boundaries
-    geometry: polygon
-    <<: *extents
-    Datasource:
-      <<: *osm2pgsql
-      table: |-
-        (SELECT
-            way,
-            name,
-            boundary,
-            way_area/NULLIF(!pixel_width!::real*!pixel_height!::real,0) AS way_pixels
-          FROM planet_osm_polygon
-          WHERE (boundary = 'national_park' OR leisure = 'nature_reserve')
-            AND building IS NULL
-            AND way_area > 0.01*!pixel_width!::real*!pixel_height!::real
-        ) AS national_park_boundaries
-    properties:
-      minzoom: 7
   - id: tourism-boundary
     name: tourism-boundary
     geometry: polygon
