--- a/admin.mss
+++ b/admin.mss
@@ -11,46 +11,62 @@
 #admin-high-zoom[zoom >= 13]*/ {
   [admin_level = '2'] {
     [zoom >= 4] {
+      /*
       background/line-join: bevel;
       background/line-color: white;
       background/line-width: 1.2;
+      */
       line-join: bevel;
       line-color: @admin-boundaries;
       line-width: 1.2;
     }
     [zoom >= 5] {
+      /*
       background/line-width: 1.5;
+      */
       line-width: 1.5;
     }
     [zoom >= 6] {
+      /*
       background/line-width: 1.8;
+      */
       line-width: 1.8;
     }
     [zoom >= 7] {
+      /*
       background/line-width: 2;
+      */
       line-width: 2;
     }
     [zoom >= 10] {
+      /*
       background/line-width: 6;
+      */
       line-width: 6;
     }
   }
 
   [admin_level = '3'] {
     [zoom >= 4] {
+      /*
       background/line-join: bevel;
       background/line-color: white;
       background/line-width: 0.6;
+      */
       line-join: bevel;
       line-color: @admin-boundaries;
       line-width: 0.6;
     }
     [zoom >= 7] {
+      /*
       background/line-width: 1.2;
+      */
       line-width: 1.2;
     }
     [zoom >= 10] {
+      /*
       background/line-width: 4;
+      */
       line-width: 4;
       line-dasharray: 4,2;
       line-clip: false;
@@ -164,8 +180,8 @@
       line-clip: false;
     }
   }
-  opacity: 0.5;
-  comp-op: darken;
+  opacity: 0.7;
+  // comp-op: darken;
 }
 */
 
