--- a/project.mml
+++ b/project.mml
@@ -97,7 +97,7 @@
       file: data/world_boundaries/builtup_area.shp
       type: shape
     properties:
-      minzoom: 8
+      minzoom: 6
       maxzoom: 9
   - id: necountries
     name: necountries
--- a/shapefiles.mss
+++ b/shapefiles.mss
@@ -35,7 +35,8 @@
 }
 
 #builtup {
-  [zoom >= 8][zoom < 10] {
+  [zoom >= 6][zoom < 10] {
     polygon-fill: #ddd;
+    polygon-opacity: 0.5;
   }
 }
