--- a/project.mml
+++ b/project.mml
@@ -1502,50 +1502,13 @@
             admin_level
           FROM planet_osm_roads
           WHERE boundary = 'administrative'
-            AND admin_level IN ('0', '1', '2', '3', '4')
+            AND admin_level IN ('2', '3')
             AND osm_id < 0
           ORDER BY admin_level DESC
         ) AS admin_low_zoom
     properties:
       minzoom: 4
       maxzoom: 10
-  - id: admin-mid-zoom
-    name: admin-mid-zoom
-    geometry: linestring
-    <<: *extents
-    Datasource:
-      <<: *osm2pgsql
-      table: |-
-        (SELECT
-            way,
-            admin_level
-          FROM planet_osm_roads
-          WHERE boundary = 'administrative'
-            AND admin_level IN ('0', '1', '2', '3', '4', '5', '6', '7', '8')
-            AND osm_id < 0
-          ORDER BY admin_level DESC
-        ) AS admin_mid_zoom
-    properties:
-      minzoom: 11
-      maxzoom: 12
-  - id: admin-high-zoom
-    name: admin-high-zoom
-    geometry: linestring
-    <<: *extents
-    Datasource:
-      <<: *osm2pgsql
-      table: |-
-        (SELECT
-            way,
-            admin_level
-          FROM planet_osm_roads
-          WHERE boundary = 'administrative'
-            AND admin_level IN ('0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '10')
-            AND osm_id < 0
-          ORDER BY admin_level::integer DESC -- With 10 as a valid value, we need to do a numeric ordering, not a text ordering
-        ) AS admin_high_zoom
-    properties:
-      minzoom: 13
   - id: tourism-boundary
     name: tourism-boundary
     geometry: polygon
--- a/admin.mss
+++ b/admin.mss
@@ -6,9 +6,9 @@
 overlapping borders correctly.
 */
 
-#admin-low-zoom[zoom < 11],
+#admin-low-zoom/*[zoom < 11],
 #admin-mid-zoom[zoom >= 11][zoom < 13],
-#admin-high-zoom[zoom >= 13] {
+#admin-high-zoom[zoom >= 13]*/ {
   [admin_level = '2'] {
     [zoom >= 4] {
       background/line-join: bevel;
@@ -57,6 +57,7 @@
     }
   }
 
+  /*
   [admin_level = '4'] {
     [zoom >= 4] {
       background/line-join: bevel;
@@ -93,6 +94,7 @@
       line-width: 3;
     }
   }
+  */
   /*
   The following code prevents admin boundaries from being rendered on top of
   each other. Comp-op works on the entire attachment, not on the individual
@@ -108,6 +110,7 @@
   comp-op: darken;
 }
 
+/*
 #admin-mid-zoom[zoom >= 11][zoom < 13],
 #admin-high-zoom[zoom >= 13] {
   [admin_level = '5'][zoom >= 11] {
@@ -164,6 +167,7 @@
   opacity: 0.5;
   comp-op: darken;
 }
+*/
 
 /*
 #admin-text[zoom >= 16] {
