--- a/style.mss
+++ b/style.mss
@@ -346,9 +346,6 @@
 @path-casing: @default-casing;
 @path-fill: black;
 
-@footway-casing: @default-casing;
-@footway-fill: salmon;
-
 @steps-casing: @default-casing;
 @steps-fill: @footway-fill;
 
@@ -366,12 +363,14 @@
 @track-grade1-fill: #b37700;
 @track-grade2-fill: #a87000;
 
+@footway-casing: @default-casing;
+@footway-fill: @track-fill;
+
 @aeroway-fill: #bbc;
 @runway-fill: @aeroway-fill;
 @taxiway-fill: @aeroway-fill;
 @helipad-fill: @aeroway-fill;
 
-
 @rail-fill: #404040;
 
 // terrain-small [0-6], terrain-medium [7-8], terrain-big [9-]
