--- a/amenity-points.mss
+++ b/amenity-points.mss
@@ -75,14 +75,14 @@
   [feature = 'highway_bus_stop'] {
     [zoom >= 16] {
       marker-file: url('symbols/square.svg');
-      marker-fill: @transportation-icon;
+      marker-fill: black;
       marker-placement: interior;
       marker-width: 6;
       marker-clip: false;
     }
     [zoom >= 17] {
       marker-file: url('symbols/bus_stop.12.svg');
-      marker-width: 12;
+      marker-width: 14;
     }
   }
 
--- a/style.mss
+++ b/style.mss
@@ -288,6 +288,7 @@
 // amenities
 @attraction: #f2caea;
 @barracks: #ff8f8f;
+@transportation-icon: @water-dark;
 
 @darken-lighten: 15%;
 
