Index: osm-carto/roads.mss
===================================================================
--- osm-carto.orig/roads.mss
+++ osm-carto/roads.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 /* For the main linear features, such as roads and railways. */
 
 //road colors for major roads were generated with scripts/generate_road_colors.py
@@ -802,8 +804,10 @@
       }
     }
 
+    /*
     [feature = 'railway_disused'][zoom >= 15],
     [feature = 'railway_construction'],
+    */
     [feature = 'railway_miniature'][zoom >= 15],
     [feature = 'railway_INT-preserved-ssy'][zoom >= 14] {
       .bridges-casing {
@@ -1015,8 +1019,10 @@
       }
     }
 
+    /*
     [feature = 'railway_disused'][zoom >= 15],
     [feature = 'railway_construction'],
+    */
     [feature = 'railway_miniature'][zoom >= 15],
     [feature = 'railway_INT-preserved-ssy'][zoom >= 14] {
       .bridges-casing {
@@ -1086,6 +1092,7 @@ tertiary is rendered from z10 and is not
      * https://github.com/mapbox/carto/issues/235
      * https://github.com/mapbox/carto/issues/237
      */
+    /*
     [feature = 'highway_construction'] {
       [zoom >= 12] {
         line-width: 2;
@@ -1161,6 +1168,7 @@ tertiary is rendered from z10 and is not
         }
       }
     }
+    */
 
     [feature = 'highway_motorway'] {
       [zoom >= 5] {
@@ -1275,20 +1283,34 @@ tertiary is rendered from z10 and is not
         }
         line-cap: round;
         line-join: round;
+
+        [int_surface = 'unpaved'] {
+          line-cap: butt; // so the dashes look 'good'
+          [zoom >= 8]  { line-dasharray: @primary-width-z8*4,  @primary-width-z8;  }
+          [zoom >= 9]  { line-dasharray: @primary-width-z9*4,  @primary-width-z9;  }
+          [zoom >= 10] { line-dasharray: @primary-width-z10*4, @primary-width-z10; }
+          [zoom >= 11] { line-dasharray: @primary-width-z11*4, @primary-width-z11; }
+          [zoom >= 12] { line-dasharray: @primary-width-z12*4, @primary-width-z12; }
+          [zoom >= 13] { line-dasharray: @primary-width-z13*4, @primary-width-z13; }
+          [zoom >= 15] { line-dasharray: @primary-width-z15*4, @primary-width-z15; }
+          [zoom >= 17] { line-dasharray: @primary-width-z17*4, @primary-width-z17; }
+          [zoom >= 18] { line-dasharray: @primary-width-z18*4, @primary-width-z18; }
+          [zoom >= 19] { line-dasharray: @primary-width-z19*4, @primary-width-z19; }
+        }
       }
     }
 
     [feature = 'highway_secondary'] {
       [zoom >= 9] {
-        line-color: @unimportant-road;
+        line-color: @secondary-low-zoom;
         line-width: @secondary-width-z9;
       }
       [zoom >=10] {
-        line-color: @unimportant-road;
+        line-color: @secondary-low-zoom;
         line-width: @secondary-width-z10;
       }
       [zoom >= 11] { line-width: @secondary-width-z11; }
-      [zoom >= 12] { 
+      [zoom >= 12] {
         line-color: @secondary-fill;
         line-width: @secondary-width-z12 - 2 * @secondary-casing-width-z12;
         line-cap: round;
@@ -1315,6 +1337,18 @@ tertiary is rendered from z10 and is not
             [zoom >= 19] { line-width: @secondary-width-z19 - 2 * @major-bridge-casing-width-z19; }
           }
         }
+
+        [int_surface = 'unpaved'] {
+          line-cap: butt; // so the dashes look 'good'
+          line-dasharray: @secondary-width-z12*4, @secondary-width-z12;
+          [zoom >= 13] { line-dasharray: @secondary-width-z13*4, @secondary-width-z13; }
+          [zoom >= 14] { line-dasharray: @secondary-width-z14*4, @secondary-width-z14; }
+          [zoom >= 15] { line-dasharray: @secondary-width-z15*4, @secondary-width-z15; }
+          [zoom >= 16] { line-dasharray: @secondary-width-z16*4, @secondary-width-z16; }
+          [zoom >= 17] { line-dasharray: @secondary-width-z17*4, @secondary-width-z17; }
+          [zoom >= 18] { line-dasharray: @secondary-width-z18*4, @secondary-width-z18; }
+          [zoom >= 19] { line-dasharray: @secondary-width-z19*4, @secondary-width-z19; }
+        }
       }
     }
 
@@ -1348,6 +1382,17 @@ tertiary is rendered from z10 and is not
         }
         line-cap: round;
         line-join: round;
+
+        [int_surface = 'unpaved'] {
+          line-dasharray: @tertiary-width-z12*4, @tertiary-width-z12;
+          [zoom >= 13] { line-dasharray: @tertiary-width-z13*4, @tertiary-width-z13; }
+          [zoom >= 14] { line-dasharray: @tertiary-width-z14*4, @tertiary-width-z14; }
+          [zoom >= 15] { line-dasharray: @tertiary-width-z15*4, @tertiary-width-z15; }
+          [zoom >= 16] { line-dasharray: @tertiary-width-z16*4, @tertiary-width-z16; }
+          [zoom >= 17] { line-dasharray: @tertiary-width-z17*4, @tertiary-width-z17; }
+          [zoom >= 18] { line-dasharray: @tertiary-width-z18*4, @tertiary-width-z18; }
+          [zoom >= 19] { line-dasharray: @tertiary-width-z19*4, @tertiary-width-z19; }
+        }
       }
     }
 
@@ -1384,11 +1429,20 @@ tertiary is rendered from z10 and is not
           [zoom >= 18] { line-width: @residential-width-z18 - 2 * @bridge-casing-width-z18; }
           [zoom >= 19] { line-width: @residential-width-z19 - 2 * @bridge-casing-width-z19; }
         }
-        line-cap: round;
+        // line-cap: round;
         line-join: round;
+
+        [zoom >= 13][int_surface = 'unpaved'] { line-dasharray: @residential-width-z13*2, @residential-width-z13; }
+        [zoom >= 14][int_surface = 'unpaved'] { line-dasharray: @residential-width-z14*2, @residential-width-z14; }
+        [zoom >= 15][int_surface = 'unpaved'] { line-dasharray: @residential-width-z15*2, @residential-width-z15; }
+        [zoom >= 16][int_surface = 'unpaved'] { line-dasharray: @residential-width-z16*2, @residential-width-z16; }
+        [zoom >= 17][int_surface = 'unpaved'] { line-dasharray: @residential-width-z17*2, @residential-width-z17; }
+        [zoom >= 18][int_surface = 'unpaved'] { line-dasharray: @residential-width-z18*2, @residential-width-z18; }
+        [zoom >= 19][int_surface = 'unpaved'] { line-dasharray: @residential-width-z19*2, @residential-width-z19; }
       }
     }
 
+
     [feature = 'highway_living_street'] {
       [zoom >= 13] {
         line-width: @living-street-width-z13 - 2 * @casing-width-z13;
@@ -1808,10 +1862,10 @@ tertiary is rendered from z10 and is not
     [feature = 'railway_rail'][zoom >= 7],
     [feature = 'railway_INT-spur-siding-yard'][zoom >= 13] {
       [zoom < 13] {
-        line-color: #787878;
-        line-width: 0.5;
-        [zoom >= 8] { line-width: 0.8; }
-        [zoom >= 12] { line-width: 0.9; }
+        line-color: @rail-fill;
+        line-width: 1;
+        [zoom >= 8] { line-width: 1.5; }
+        [zoom >= 11] { line-width: 2; }
         line-join: round;
         .roads_low_zoom[int_tunnel = 'yes'], .tunnels-fill {
           line-dasharray: 5,2;
@@ -1823,7 +1877,7 @@ tertiary is rendered from z10 and is not
           light/line-color: white;
           light/line-join: round;
           [feature = 'railway_rail'] {
-            dark/line-color: #787878;
+            dark/line-color: @rail-fill;
             dark/line-width: 3;
             light/line-width: 1;
             light/line-dasharray: 8,8;
@@ -1987,6 +2041,7 @@ tertiary is rendered from z10 and is not
       }
     }
 
+    /*
     [feature = 'railway_construction'] {
       [zoom >= 13] {
         line-color: grey;
@@ -2002,7 +2057,9 @@ tertiary is rendered from z10 and is not
         }
       }
     }
+    */
 
+    /*
     [feature = 'railway_disused'] {
       [zoom >= 15] {
         line-color: #aaa;
@@ -2011,6 +2068,7 @@ tertiary is rendered from z10 and is not
         line-join: round;
       }
     }
+    */
 
     [feature = 'railway_platform'] {
       [zoom >= 16] {
@@ -2055,6 +2113,23 @@ tertiary is rendered from z10 and is not
       }
     }
   }
+
+  ::strip {
+    [feature = 'highway_motorway'][link = 'no'],
+    [feature = 'highway_trunk'][link = 'no'] {
+      [zoom >= 12] {
+        line-width: 1;
+        [feature = 'highway_motorway'] {
+          line-color: yellow;
+        }
+        [feature = 'highway_trunk'][link = 'no'] {
+          line-color: white;
+        }
+      }
+      [zoom >= 14] { line-width: 1.5; }
+      [zoom >= 16] { line-width: 2; }
+    }
+  }
 }
 
 #turning-circle-casing {
@@ -2547,7 +2622,7 @@ tertiary is rendered from z10 and is not
         access/line-opacity: 0.5;
         access/line-join: round;
         access/line-cap: round;
-      }    
+      }
     }
 
     [feature = 'highway_service'][service = 'INT-minor'][zoom >= 16] {
@@ -2740,25 +2815,31 @@ tertiary is rendered from z10 and is not
   [highway = 'primary'] {
     [zoom >= 13] {
       text-name: "[name]";
-      text-size: 8;
+      text-size: 10;
       text-fill: black;
       text-spacing: 300;
       text-clip: false;
       text-placement: line;
       text-face-name: @book-fonts;
       [tunnel = 'no'] {
+        /*
         text-halo-radius: 1;
         [highway = 'motorway'] { text-halo-fill: @motorway-fill; }
         [highway = 'trunk'] { text-halo-fill: @trunk-fill; }
         [highway = 'primary'] { text-halo-fill: @primary-fill; }
+        */
+        text-halo-radius: 1.5;
+        text-halo-fill: white;
       }
     }
+    /*
     [zoom >= 14] {
       text-size: 9;
     }
     [zoom >= 15] {
       text-size: 10;
     }
+    */
     [zoom >= 17] {
       text-size: 11;
     }
@@ -2769,21 +2850,23 @@ tertiary is rendered from z10 and is not
   [highway = 'secondary'] {
     [zoom >= 13] {
       text-name: "[name]";
-      text-size: 8;
+      text-size: 10;
       text-fill: black;
       text-spacing: 300;
       text-clip: false;
       text-placement: line;
       text-face-name: @book-fonts;
-      text-halo-radius: 1;
-      text-halo-fill: @secondary-fill;
+      text-halo-radius: 1.5;
+      text-halo-fill: white;
     }
+    /*
     [zoom >= 14] {
       text-size: 9;
     }
     [zoom >= 15] {
       text-size: 10;
     }
+    */
     [zoom >= 17] {
       text-size: 11;
     }
@@ -2795,14 +2878,14 @@ tertiary is rendered from z10 and is not
   [highway = 'tertiary_link'] {
     [zoom >= 14] {
       text-name: "[name]";
-      text-size: 9;
+      text-size: 10;
       text-fill: black;
       text-spacing: 300;
       text-clip: false;
       text-placement: line;
       text-face-name: @book-fonts;
-      text-halo-radius: 1;
-      text-halo-fill: @tertiary-fill;
+      text-halo-radius: 1.5;
+      text-halo-fill: white;
     }
     [zoom >= 17] {
       text-size: 11;
@@ -2811,6 +2894,7 @@ tertiary is rendered from z10 and is not
       text-size: 12;
     }
   }
+  /*
   [highway = 'construction'] {
     [int_construction_minor = 'no'][zoom >= 13],
     [int_construction_minor = 'yes'][zoom >= 14] {
@@ -2831,23 +2915,26 @@ tertiary is rendered from z10 and is not
       }
     }
   }
+  */
   [highway = 'residential'],
   [highway = 'unclassified'],
   [highway = 'road'] {
     [zoom >= 15] {
       text-name: "[name]";
-      text-size: 8;
+      text-size: 10;
       text-fill: black;
       text-spacing: 300;
       text-clip: false;
       text-placement: line;
       text-halo-radius: 1;
-      text-halo-fill: @residential-fill;
+      text-halo-fill: white;
       text-face-name: @book-fonts;
     }
+    /*
     [zoom >= 16] {
       text-size: 9;
     }
+    */
     [zoom >= 17] {
       text-size: 11;
       text-spacing: 400;
@@ -3001,7 +3088,7 @@ tertiary is rendered from z10 and is not
       [oneway = 'yes'],
       [oneway = '-1'] {
         marker-placement: line;
-        marker-spacing: 180;
+        marker-spacing: 50;
         marker-max-error: 0.5;
 
         marker-file: url('symbols/oneway.svg');
