--- a/landcover.mss
+++ b/landcover.mss
@@ -119,6 +119,8 @@
     [way_pixels >= 64] { polygon-pattern-gamma: 0.3;  }
   }
 
+  // TODO: make them transparent
+  /*
   [feature = 'landuse_vineyard'] {
     [zoom >= 10] {
       polygon-fill: @orchard;
@@ -146,6 +148,7 @@
       [way_pixels >= 64] { polygon-pattern-gamma: 0.3;  }
     }
   }
+  */
 
   [feature = 'landuse_cemetery'],
   [feature = 'amenity_grave_yard'] {
@@ -248,6 +251,7 @@
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
   }
 
+  /*
   [feature = 'landuse_allotments'] {
     [zoom >= 10] {
       polygon-fill: @allotments;
@@ -261,6 +265,7 @@
       [way_pixels >= 64] { polygon-pattern-gamma: 0.3;  }
     }
   }
+  */
 
   [feature = 'landuse_forest'],
   [feature = 'natural_wood'] {
@@ -271,6 +276,7 @@
     }
   }
 
+  /*
   [feature = 'landuse_farmyard'][zoom >= 10] {
     polygon-fill: @farmyard;
       [zoom >= 16] {
@@ -283,7 +289,9 @@
     [way_pixels >= 4]  { polygon-gamma: 0.75; }
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
   }
+  */
 
+  /*
   [feature = 'landuse_farm'],
   [feature = 'landuse_farmland'],
   [feature = 'landuse_greenhouse_horticulture'] {
@@ -297,10 +305,13 @@
       [way_pixels >= 64] { polygon-gamma: 0.3;  }
     }
   }
+  */
 
+  /*
   [feature = 'landuse_meadow'],
   [feature = 'natural_grassland'],
   [feature = 'landuse_grass'],
+  /*
   [feature = 'landuse_recreation_ground'],
   [feature = 'landuse_village_green'],
   [feature = 'leisure_common'],
@@ -312,6 +323,7 @@
     }
   }
 
+  /*
   [feature = 'landuse_retail'][zoom >= 10] {
     polygon-fill: @retail;
     [zoom >= 16] {
@@ -324,7 +336,9 @@
       [way_pixels >= 64] { polygon-gamma: 0.3;  }
     }
   }
+  */
 
+  /*
   [feature = 'landuse_industrial'][zoom >= 10] {
     polygon-fill: @industrial;
     [zoom >= 16] {
@@ -337,6 +351,7 @@
     [way_pixels >= 4]  { polygon-gamma: 0.75; }
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
   }
+  */
 
   [feature = 'landuse_railway'][zoom >= 10] {
     polygon-fill: @railway;
@@ -348,6 +363,7 @@
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
   }
 
+  /*
   [feature = 'power_station'][zoom >= 10],
   [feature = 'power_generator'][zoom >= 10],
   [feature = 'power_sub_station'][zoom >= 13],
@@ -366,6 +382,7 @@
     [way_pixels >= 4]  { polygon-gamma: 0.75; }
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
   }
+  */
 
   [feature = 'landuse_commercial'][zoom >= 10] {
     polygon-fill: @commercial;
@@ -380,6 +397,7 @@
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
   }
 
+  /*
   [feature = 'landuse_brownfield'],
   [feature = 'landuse_construction'] {
     [zoom >= 10] {
@@ -396,6 +414,7 @@
       [way_pixels >= 64] { polygon-gamma: 0.3;  }
     }
   }
+  */
 
   [feature = 'natural_bare_rock'][zoom >= 9] {
     polygon-fill: @bare_ground;
