--- a/amenity-points.mss
+++ b/amenity-points.mss
@@ -2190,6 +2190,7 @@
     text-wrap-width: @standard-wrap-width;
   }
 
+  /*
   [feature = 'natural_tree'][zoom >= 17] {
     text-name: "[name]";
     text-size: @standard-text-size;
@@ -2204,8 +2205,10 @@
     text-placement: interior;
     text-wrap-width: @standard-wrap-width;
   }
+  */
 }
 
+/*
 #trees [zoom >= 16] {
   ::canopy {
     opacity: 0.3;
@@ -2270,3 +2273,4 @@
     }
   }
 }
+*/
--- a/project.mml
+++ b/project.mml
@@ -1394,25 +1394,6 @@
         ) AS tourism_boundary
     properties:
       minzoom: 10
-  - id: trees
-    name: trees
-    geometry: polygon
-    <<: *extents
-    Datasource:
-      <<: *osm2pgsql
-      table: |-
-        (SELECT
-            way, "natural"
-          FROM planet_osm_point
-          WHERE "natural" = 'tree'
-        UNION ALL
-        SELECT
-            way, "natural"
-          FROM planet_osm_line
-          WHERE "natural" = 'tree_row'
-        ) AS trees
-    properties:
-      minzoom: 16
   - id: country-names
     name: country-names
     class: country
