--- a/project.mml
+++ b/project.mml
@@ -139,7 +139,7 @@
               ('landuse_' || (CASE WHEN landuse IN ('quarry', 'vineyard', 'orchard', 'cemetery', 'residential', 'garages', 'meadow', 'grass', 
                                                     'allotments', 'forest', 'farmyard', 'farm', 'farmland', 'greenhouse_horticulture', 
                                                     'recreation_ground', 'village_green', 'retail', 'industrial', 'railway', 'commercial', 
-                                                    'brownfield', 'landfill', 'construction') THEN landuse ELSE NULL END)) AS landuse,
+                                                    'brownfield', 'landfill') THEN landuse ELSE NULL END)) AS landuse,
               ('leisure_' || (CASE WHEN leisure IN ('swimming_pool', 'playground', 'park', 'recreation_ground', 'common', 'garden', 
                                                     'golf_course', 'miniature_golf', 'picnic_table', 'sports_centre', 'stadium', 'pitch', 
                                                     'track', 'dog_park') THEN leisure ELSE NULL END)) AS leisure,
@@ -474,7 +474,6 @@
             tracktype,
             int_surface,
             access,
-            construction,
             service,
             link,
             layernotnull
@@ -496,7 +495,6 @@
                   WHEN access IN ('no', 'private') THEN 'no'::text
                   ELSE NULL
                 END AS access,
-                construction,
                 CASE
                   WHEN service IN ('parking_aisle', 'drive-through', 'driveway') THEN 'INT-minor'::text
                   ELSE 'INT-normal'::text
@@ -528,7 +526,6 @@
                   WHEN access IN ('no', 'private') THEN 'no'::text
                   ELSE NULL
                 END AS access,
-                construction,
                 CASE WHEN service IN ('parking_aisle', 'drive-through', 'driveway') THEN 'INT-minor'::text ELSE 'INT-normal'::text END AS service,
                 'no' AS link,
                 CASE WHEN layer~E'^-?\\d+$' AND length(layer)<10 THEN layer::integer ELSE 0 END AS layernotnull
@@ -551,7 +548,6 @@
               ('railway_tram', 410),
               ('railway_tram-service', 405),
               ('railway_disused', 400),
-              ('railway_construction', 400),
               ('highway_motorway', 380),
               ('highway_trunk', 370),
               ('highway_primary', 360),
@@ -578,8 +574,7 @@
               ('highway_platform', 90),
               ('railway_platform', 90),
               ('aeroway_runway', 60),
-              ('aeroway_taxiway', 50),
-              ('highway_construction', 10)
+              ('aeroway_taxiway', 50)
             ) AS ordertable (feature, prio)
             USING (feature)
           ORDER BY
@@ -743,7 +738,6 @@
             tracktype,
             int_surface,
             access,
-            construction,
             service,
             link,
             layernotnull
@@ -765,7 +759,6 @@
                   WHEN access IN ('no', 'private') THEN 'no'::text
                   ELSE NULL
                 END AS access,
-                construction,
                 CASE
                   WHEN service IN ('parking_aisle', 'drive-through', 'driveway') THEN 'INT-minor'::text
                   ELSE 'INT-normal'::text
@@ -799,7 +792,6 @@
                   WHEN access IN ('no', 'private') THEN 'no'::text
                   ELSE NULL
                 END AS access,
-                construction,
                 CASE WHEN service IN ('parking_aisle', 'drive-through', 'driveway') THEN 'INT-minor'::text ELSE 'INT-normal'::text END AS service,
                 'no' AS link,
                 CASE WHEN layer~E'^-?\\d+$' AND length(layer)<10 THEN layer::integer ELSE 0 END AS layernotnull
@@ -824,7 +816,6 @@
               ('railway_tram', 410),
               ('railway_tram-service', 405),
               ('railway_disused', 400),
-              ('railway_construction', 400),
               ('highway_motorway', 380),
               ('highway_trunk', 370),
               ('highway_primary', 360),
@@ -851,8 +842,7 @@
               ('highway_platform', 90),
               ('railway_platform', 90),
               ('aeroway_runway', 60),
-              ('aeroway_taxiway', 50),
-              ('highway_construction', 10)
+              ('aeroway_taxiway', 50)
             ) AS ordertable (feature, prio)
             USING (feature)
           ORDER BY
@@ -908,7 +898,6 @@
             tracktype,
             int_surface,
             access,
-            construction,
             service,
             link,
             layernotnull
@@ -930,7 +919,6 @@
                   WHEN access IN ('no', 'private') THEN 'no'::text
                   ELSE NULL
                 END AS access,
-                construction,
                 CASE
                   WHEN service IN ('parking_aisle', 'drive-through', 'driveway') THEN 'INT-minor'::text
                   ELSE 'INT-normal'::text
@@ -964,7 +952,6 @@
                   WHEN access IN ('no', 'private') THEN 'no'::text
                   ELSE NULL
                 END AS access,
-                construction,
                 CASE WHEN service IN ('parking_aisle', 'drive-through', 'driveway') THEN 'INT-minor'::text ELSE 'INT-normal'::text END AS service,
                 'no' AS link,
                 CASE WHEN layer~E'^-?\\d+$' AND length(layer)<10 THEN layer::integer ELSE 0 END AS layernotnull
@@ -989,7 +976,6 @@
               ('railway_tram', 410),
               ('railway_tram-service', 405),
               ('railway_disused', 400),
-              ('railway_construction', 400),
               ('highway_motorway', 380),
               ('highway_trunk', 370),
               ('highway_primary', 360),
@@ -1016,8 +1002,7 @@
               ('highway_platform', 90),
               ('railway_platform', 90),
               ('aeroway_runway', 60),
-              ('aeroway_taxiway', 50),
-              ('highway_construction', 10)
+              ('aeroway_taxiway', 50)
             ) AS ordertable (feature, prio)
             USING (feature)
           ORDER BY
@@ -1142,7 +1127,6 @@
             tracktype,
             int_surface,
             access,
-            construction,
             service,
             link,
             layernotnull
@@ -1164,7 +1148,6 @@
                   WHEN access IN ('no', 'private') THEN 'no'::text
                   ELSE NULL
                 END AS access,
-                construction,
                 CASE
                   WHEN service IN ('parking_aisle', 'drive-through', 'driveway') THEN 'INT-minor'::text
                   ELSE 'INT-normal'::text
@@ -1196,7 +1179,6 @@
                   WHEN access IN ('no', 'private') THEN 'no'::text
                   ELSE NULL
                 END AS access,
-                construction,
                 CASE WHEN service IN ('parking_aisle', 'drive-through', 'driveway') THEN 'INT-minor'::text ELSE 'INT-normal'::text END AS service,
                 'no' AS link,
                 CASE WHEN layer~E'^-?\\d+$' AND length(layer)<10 THEN layer::integer ELSE 0 END AS layernotnull
@@ -1219,7 +1201,6 @@
               ('railway_tram', 410),
               ('railway_tram-service', 405),
               ('railway_disused', 400),
-              ('railway_construction', 400),
               ('highway_motorway', 380),
               ('highway_trunk', 370),
               ('highway_primary', 360),
@@ -1246,8 +1227,7 @@
               ('highway_platform', 90),
               ('railway_platform', 90),
               ('aeroway_runway', 60),
-              ('aeroway_taxiway', 50),
-              ('highway_construction', 10)
+              ('aeroway_taxiway', 50)
             ) AS ordertable (feature, prio)
             USING (feature)
           ORDER BY
@@ -1878,7 +1858,6 @@
             way,
             CASE WHEN substr(highway, length(highway)-3, 4) = 'link' THEN substr(highway, 0, length(highway)-4) ELSE highway END,
             CASE WHEN (tunnel = 'yes' OR tunnel = 'building_passage' OR covered = 'yes') THEN 'yes' ELSE 'no' END AS tunnel,
-            CASE WHEN construction IN ('service', 'footway', 'cycleway', 'bridleway', 'path', 'track') THEN 'yes' ELSE 'no' END AS int_construction_minor,
             name,
             CASE
               WHEN oneway IN ('yes', '-1') THEN oneway
@@ -1909,7 +1888,7 @@
             ) AS ordertable (highway, prio)
             USING (highway)
           WHERE highway IN ('motorway', 'motorway_link', 'trunk', 'trunk_link', 'primary', 'primary_link', 'secondary', 'secondary_link', 'tertiary', 
-                            'tertiary_link', 'residential', 'unclassified', 'road', 'service', 'pedestrian', 'raceway', 'living_street', 'construction')
+                            'tertiary_link', 'residential', 'unclassified', 'road', 'service', 'pedestrian', 'raceway', 'living_street')
             AND (name IS NOT NULL
               OR oneway IN ('yes', '-1')
               OR junction IN ('roundabout'))
@@ -2013,7 +1992,7 @@
               'landuse_' || CASE WHEN landuse IN ('reservoir', 'basin', 'recreation_ground', 'village_green', 'quarry', 'vineyard', 'orchard', 'cemetery', 
                                                   'residential', 'garages', 'meadow', 'grass', 'allotments', 'forest', 'farmyard', 'farm', 'farmland', 
                                                   'greenhouse_horticulture', 'retail', 'industrial', 'railway', 'commercial', 'brownfield', 'landfill', 
-                                                  'construction', 'military') THEN landuse ELSE NULL END,
+                                                  'military') THEN landuse ELSE NULL END,
               'man_made_' || CASE WHEN man_made IN ('lighthouse', 'windmill', 'mast', 'water_tower', 'pier', 'breakwater', 'groyne', 'obelisk') THEN man_made ELSE NULL END,
               'natural_' || CASE WHEN "natural" IN ('wood', 'water', 'mud', 'wetland', 'marsh', 'bay', 'spring', 'scree', 'shingle', 'bare_rock', 'sand', 'heath', 
                                                     'grassland', 'scrub', 'beach', 'shoal', 'reef', 'glacier') THEN "natural" ELSE NULL END,
@@ -2140,7 +2119,7 @@
                   'landuse_' || CASE WHEN landuse IN ('reservoir', 'basin', 'recreation_ground', 'village_green', 'quarry', 'vineyard', 'orchard', 'cemetery', 
                                                       'residential', 'garages', 'meadow', 'grass', 'allotments', 'forest', 'farmyard', 'farm', 'farmland', 
                                                       'greenhouse_horticulture', 'retail', 'industrial', 'railway', 'commercial', 'brownfield', 'landfill', 
-                                                      'construction', 'military') THEN landuse ELSE NULL END,
+                                                      'military') THEN landuse ELSE NULL END,
                   'man_made_' || CASE WHEN man_made IN ('lighthouse', 'windmill', 'mast', 'water_tower', 'obelisk') THEN man_made ELSE NULL END,
                   'natural_' || CASE WHEN "natural" IN ('wood', 'peak', 'volcano', 'saddle', 'cave_entrance', 'water', 'mud', 'wetland', 'marsh', 'bay', 'spring', 
                                                         'scree', 'shingle', 'bare_rock', 'sand', 'heath', 'grassland', 'scrub', 'beach', 'glacier', 'tree') 
@@ -2180,7 +2159,7 @@
                   OR leisure IS NOT NULL
                   OR landuse IN ('reservoir', 'basin', 'recreation_ground', 'village_green', 'quarry', 'vineyard', 'orchard', 'cemetery', 'residential', 
                                  'garages', 'meadow', 'grass', 'allotments', 'forest', 'farmyard', 'farm', 'farmland', 'greenhouse_horticulture', 
-                                 'retail', 'industrial', 'railway', 'commercial', 'brownfield', 'landfill', 'construction', 'military')
+                                 'retail', 'industrial', 'railway', 'commercial', 'brownfield', 'landfill', 'military')
                   OR man_made IN ('lighthouse', 'windmill', 'mast', 'water_tower', 'cross', 'obelisk')
                   OR "natural" IS NOT NULL
                   OR place IN ('island', 'islet')
--- a/roads.mss
+++ b/roads.mss
@@ -789,7 +789,7 @@
     }
 
     [feature = 'railway_disused'][zoom >= 15],
-    [feature = 'railway_construction'],
+    // [feature = 'railway_construction'],
     [feature = 'railway_miniature'][zoom >= 15],
     [feature = 'railway_INT-preserved-ssy'][zoom >= 14] {
       .bridges-casing {
@@ -1074,6 +1074,7 @@
      * https://github.com/mapbox/carto/issues/235
      * https://github.com/mapbox/carto/issues/237
      */
+    /*
     [feature = 'highway_construction'] {
       [zoom >= 12] {
         line-width: 2;
@@ -1149,6 +1150,7 @@
         }
       }
     }
+    */
 
     [feature = 'highway_motorway'] {
       [zoom >= 5] {
@@ -1991,6 +1993,7 @@
       }
     }
 
+    /*
     [feature = 'railway_construction'] {
       [zoom >= 13] {
         line-color: grey;
@@ -2006,6 +2009,7 @@
         }
       }
     }
+    */
 
     [feature = 'railway_disused'] {
       [zoom >= 15] {
@@ -2815,6 +2819,7 @@
       text-size: 12;
     }
   }
+  /*
   [highway = 'construction'] {
     [int_construction_minor = 'no'][zoom >= 13],
     [int_construction_minor = 'yes'][zoom >= 14] {
@@ -2835,6 +2840,7 @@
       }
     }
   }
+  */
   [highway = 'residential'],
   [highway = 'unclassified'],
   [highway = 'road'] {
