--- a/power.mss
+++ b/power.mss
@@ -1,5 +1,6 @@
 @power-line-color: #888;
 
+/*
 #power-line {
   [zoom >= 14] {
     line-width: 0.5;
@@ -53,3 +54,4 @@
     marker-width: 3;
   }
 }
+*/
--- a/project.mml
+++ b/project.mml
@@ -1329,34 +1329,6 @@
         ) AS admin_high_zoom
     properties:
       minzoom: 13
-  - id: power-minorline
-    name: power-minorline
-    geometry: linestring
-    <<: *extents
-    Datasource:
-      <<: *osm2pgsql
-      table: |-
-        (SELECT
-            way
-          FROM planet_osm_line
-          WHERE power = 'minor_line'
-        ) AS power_minorline
-    properties:
-      minzoom: 16
-  - id: power-line
-    name: power-line
-    geometry: linestring
-    <<: *extents
-    Datasource:
-      <<: *osm2pgsql
-      table: |-
-        (SELECT
-            way
-          FROM planet_osm_line
-          WHERE power = 'line'
-        ) AS power_line
-    properties:
-      minzoom: 14
   - id: nature-reserve-boundaries
     name: nature-reserve-boundaries
     geometry: polygon
@@ -1731,34 +1703,6 @@
           ) AS amenity_points
     properties:
       minzoom: 10
-  - id: power-towers
-    name: power-towers
-    geometry: point
-    <<: *extents
-    Datasource:
-      <<: *osm2pgsql
-      table: |-
-        (SELECT
-            way
-          FROM planet_osm_point
-          WHERE power = 'tower'
-        ) AS power_towers
-    properties:
-      minzoom: 14
-  - id: power-poles
-    name: power-poles
-    geometry: point
-    <<: *extents
-    Datasource:
-      <<: *osm2pgsql
-      table: |-
-        (SELECT
-            way
-          FROM planet_osm_point
-          WHERE power = 'pole'
-        ) AS power_poles
-    properties:
-      minzoom: 16
   - id: roads-text-ref-low-zoom
     name: roads-text-ref-low-zoom
     geometry: linestring
