--- a/project.mml
+++ b/project.mml
@@ -547,7 +547,6 @@
               ('railway_turntable', 420),
               ('railway_tram', 410),
               ('railway_tram-service', 405),
-              ('railway_disused', 400),
               ('highway_motorway', 380),
               ('highway_trunk', 370),
               ('highway_primary', 360),
@@ -815,7 +814,6 @@
               ('railway_turntable', 420),
               ('railway_tram', 410),
               ('railway_tram-service', 405),
-              ('railway_disused', 400),
               ('highway_motorway', 380),
               ('highway_trunk', 370),
               ('highway_primary', 360),
@@ -975,7 +973,6 @@
               ('railway_turntable', 420),
               ('railway_tram', 410),
               ('railway_tram-service', 405),
-              ('railway_disused', 400),
               ('highway_motorway', 380),
               ('highway_trunk', 370),
               ('highway_primary', 360),
@@ -1200,7 +1197,6 @@
               ('railway_turntable', 420),
               ('railway_tram', 410),
               ('railway_tram-service', 405),
-              ('railway_disused', 400),
               ('highway_motorway', 380),
               ('highway_trunk', 370),
               ('highway_primary', 360),
--- a/roads.mss
+++ b/roads.mss
@@ -1003,8 +1003,10 @@
       }
     }
 
+    /*
     [feature = 'railway_disused'][zoom >= 15],
     [feature = 'railway_construction'],
+    */
     [feature = 'railway_miniature'][zoom >= 15],
     [feature = 'railway_INT-preserved-ssy'][zoom >= 14] {
       .bridges-casing {
@@ -2011,6 +2013,7 @@
     }
     */
 
+    /*
     [feature = 'railway_disused'] {
       [zoom >= 15] {
         line-color: #aaa;
@@ -2019,6 +2022,7 @@
         line-join: round;
       }
     }
+    */
 
     [feature = 'railway_platform'] {
       [zoom >= 16] {
