--- a/addressing.mss
+++ b/addressing.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 #interpolation {
   [zoom >= 17] {
     line-color: #888;
--- a/admin.mss
+++ b/admin.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 @admin-boundaries: #ac46ac;
 
 /* For performance reasons, the admin border layers are split into three groups
--- a/aerialways.mss
+++ b/aerialways.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 #aerialways {
   [aerialway = 'cable_car'],
   [aerialway = 'gondola'] {
--- a/amenity-points.mss
+++ b/amenity-points.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 @marina-text: #576ddf; // also swimming_pool
 @wetland-text: darken(#4aa5fa, 25%); /* Also for marsh and mud */
 @shop-icon: #ac39ac;
--- a/buildings.mss
+++ b/buildings.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 @building-fill: #d9d0c9; //Lch(84, 5, 70)
 @building-line: darken(@building-fill, 15%);
 @building-low-zoom: darken(@building-fill, 4%);
--- a/citywalls.mss
+++ b/citywalls.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 #citywalls {
   [zoom >= 14] {
     line-color: grey;
--- a/ferry-routes.mss
+++ b/ferry-routes.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 @ferry-route: #66f;
 @ferry-route-text: @ferry-route;
 
--- a/landcover.mss
+++ b/landcover.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 // --- Parks, woods, other green things ---
 
 @grass: #cdebb0; // also meadow, common, garden, village_green
--- a/placenames.mss
+++ b/placenames.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 @placenames: #222;
 @placenames-light: #777777;
 
--- a/power.mss
+++ b/power.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 @power-line-color: #888;
 
 /*
--- a/project.yaml
+++ b/project.yaml
@@ -1,3 +1,5 @@
+# kate: indent-width 2;
+
 scale: 1
 metatile: 2
 name: "OpenStreetMap Carto"
--- a/roads.mss
+++ b/roads.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 /* For the main linear features, such as roads and railways. */
 
 //road colors for major roads were generated with scripts/generate_road_colors.py
--- a/shapefiles.mss
+++ b/shapefiles.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 #necountries {
   [zoom >= 1][zoom < 4] {
     line-width: 0.5;
--- a/stations.mss
+++ b/stations.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 @station-color: #7981b0;
 @station-text: darken(saturate(@station-color, 15%), 10%);
 
--- a/style.mss
+++ b/style.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 Map {
   background-color: @water-color;
 }
--- a/water-features.mss
+++ b/water-features.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 @breakwater-color: #aaa; /* Also for groyne */
 @dam: #adadad;
 @dam-line: #444444;
--- a/water.mss
+++ b/water.mss
@@ -1,3 +1,5 @@
+// kate: indent-width 2;
+
 @water-text: #6699cc;
 @glacier: #ddecec;
 @glacier-line: #9cf;
