--- a/landcover.mss
+++ b/landcover.mss
@@ -130,7 +130,10 @@
       [way_pixels >= 64] { polygon-pattern-gamma: 0.3;  }
     }
   }
+  */
 
+  // TODO: make them transparent
+  /*
   [feature = 'landuse_orchard'] {
     [zoom >= 10] {
       polygon-fill: @orchard;
@@ -233,6 +236,7 @@
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
   }
 
+  // TODO: make them transparent
   /*
   [feature = 'landuse_allotments'] {
     [zoom >= 10][zoom < 14] {
@@ -249,6 +253,8 @@
   }
   */
 
+  // TODO: make them transparent
+  /*
   [feature = 'landuse_forest'],
   [feature = 'natural_wood'] {
     [zoom >= 8] {
@@ -257,7 +263,9 @@
       [way_pixels >= 64] { polygon-gamma: 0.3;  }
     }
   }
+  */
 
+  // TODO: make them transparent
   /*
   [feature = 'landuse_farmyard'][zoom >= 10] {
     polygon-fill: @farmyard;
@@ -273,6 +281,7 @@
   }
   */
 
+  // TODO: make them transparent
   /*
   [feature = 'landuse_farm'],
   [feature = 'landuse_farmland'],
@@ -289,11 +298,12 @@
   }
   */
 
+  // TODO: make them transparent
   /*
   [feature = 'landuse_meadow'],
   [feature = 'natural_grassland'],
   [feature = 'landuse_grass'],
-  /*
+  */
   [feature = 'landuse_recreation_ground'],
   [feature = 'landuse_village_green'],
   [feature = 'leisure_common'],
@@ -379,9 +389,12 @@
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
   }
 
+  // TODO: make them transparent
   /*
   [feature = 'landuse_brownfield'],
   [feature = 'landuse_landfill'],
+  */
+  /*
   [feature = 'landuse_construction'] {
     [zoom >= 10] {
       polygon-fill: @construction;
@@ -392,9 +405,11 @@
   */
 
   [feature = 'natural_bare_rock'][zoom >= 9] {
+    /*
     polygon-fill: @bare_ground;
     [way_pixels >= 4]  { polygon-gamma: 0.75; }
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
+    */
     [zoom >= 13] {
       polygon-pattern-file: url('symbols/rock_overlay.png');
       [way_pixels >= 4]  { polygon-pattern-gamma: 0.75; }
@@ -405,9 +420,11 @@
   [feature = 'natural_scree'],
   [feature = 'natural_shingle'] {
     [zoom >= 9] {
+      /*
       polygon-fill: @bare_ground;
       [way_pixels >= 4]  { polygon-gamma: 0.75; }
       [way_pixels >= 64] { polygon-gamma: 0.3;  }
+      */
       [zoom >= 13] {
         polygon-pattern-file: url('symbols/scree_overlay.png');
         [way_pixels >= 4]  { polygon-pattern-gamma: 0.75; }
@@ -422,20 +439,26 @@
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
   }
 
+  // TODO: make them transparent
+  /*
   [feature = 'natural_heath'][zoom >= 10] {
     polygon-fill: @heath;
     [way_pixels >= 4]  { polygon-gamma: 0.75; }
     [way_pixels >= 64] { polygon-gamma: 0.3;  }
   }
+  */
 
   [feature = 'natural_scrub'] {
+    /*
     [zoom >= 10] {
       polygon-fill: @scrub;
       [way_pixels >= 4]  { polygon-gamma: 0.75; }
       [way_pixels >= 64] { polygon-gamma: 0.3;  }
     }
+    */
     [zoom >= 14] {
-      polygon-pattern-file: url('symbols/scrub.png');
+      // polygon-pattern-file: url('symbols/scrub.png');
+      polygon-pattern-file: url('symbols/scrub_trans.png');
       [way_pixels >= 4]  { polygon-pattern-gamma: 0.75; }
       [way_pixels >= 64] { polygon-pattern-gamma: 0.3;  }
     }
@@ -607,9 +630,10 @@
   }
   //Also landuse = forest, converted in the SQL
   [natural = 'wood'][zoom >= 13]::wood {
-    polygon-pattern-file: url('symbols/forest.png'); // Lch(55,30,135)
+    // polygon-pattern-file: url('symbols/forest.png'); // Lch(55,30,135)
+    polygon-pattern-file: url('symbols/forest_trans.png');
     polygon-pattern-alignment: global;
-    opacity: 0.4; // The entire layer has opacity to handle overlapping forests
+    // opacity: 0.4; // The entire layer has opacity to handle overlapping forests
   }
 }
 
@@ -627,9 +651,11 @@
 
 #cliffs {
   [natural = 'cliff'][zoom >= 13] {
-    line-pattern-file: url('symbols/cliff.png');
+    // line-pattern-file: url('symbols/cliff.png');
+    line-pattern-file: url('symbols/cliff_mdione.png');
     [zoom >= 15] {
-      line-pattern-file: url('symbols/cliff2.png');
+      // line-pattern-file: url('symbols/cliff2.png');
+      line-pattern-file: url('symbols/cliff2_mdione.png');
     }
   }
   [man_made = 'embankment'][zoom >= 15]::man_made {
