--- a/roads.mss
+++ b/roads.mss
@@ -1806,13 +1806,13 @@
       }
     }
 
-    [feature = 'railway_rail'][zoom >= 7],
+    [feature = 'railway_rail'][zoom >= 5],
     [feature = 'railway_INT-spur-siding-yard'][zoom >= 13] {
       [zoom < 13] {
-        line-color: #787878;
+        line-color: @rail-fill;
         line-width: 0.5;
-        [zoom >= 8] { line-width: 0.8; }
-        [zoom >= 12] { line-width: 0.9; }
+        [zoom >= 7] { line-width: 1.0; }
+        [zoom >= 9] { line-width: 2.0; }
         line-join: round;
         .roads_low_zoom[int_tunnel = 'yes'], .tunnels-fill {
           line-dasharray: 5,2;
--- a/style.mss
+++ b/style.mss
@@ -372,6 +372,8 @@
 @helipad-fill: @aeroway-fill;
 
 
+@rail-fill: #404040;
+
 // terrain-small [0-6], terrain-medium [7-8], terrain-big [9-]
 .terrain {
   raster-scaling: lanczos;
