--- a/project.mml
+++ b/project.mml
@@ -2313,7 +2313,7 @@
                                                         'scree', 'shingle', 'bare_rock', 'sand', 'heath', 'grassland', 'scrub', 'beach', 'cliff', 'valley',
                                                         'ridge', 'arete', 'dale', 'gorge', 'couloir', 'mountain_range', 'massif', 'glacier', 'tree')
                                                         THEN "natural" ELSE NULL END,
-                  'place_' || CASE WHEN place IN ('island', 'islet') THEN place
+                  'place_' || CASE WHEN place IN ('island', 'islet', 'ocean', 'sea') THEN place
                                   ELSE NULL END,
                   'military_' || CASE WHEN military IN ('danger_area') THEN military ELSE NULL END,
                   'historic_' || CASE WHEN historic IN ('memorial', 'monument', 'archaeological_site', 'castle', 'fort') THEN historic ELSE NULL END,
--- a/water.mss
+++ b/water.mss
@@ -331,7 +331,19 @@
   }
 }
 
-.text[zoom >= 10] {
+.text[zoom >= 5] {
+  [feature = 'place_ocean'],
+  [feature = 'place_sea'] {
+    #text-point[zoom >= 5] {
+      text-name: "[name]";
+      text-halo-radius: 1;
+      text-halo-fill: rgba(255,255,255,0.6);
+      text-fill: @water-text;
+      text-size: 12;
+      text-face-name: @oblique-fonts;
+    }
+  }
+
   [feature = 'natural_water'],
   [feature = 'landuse_reservoir'],
   [feature = 'landuse_basin'] {
