--- a/placenames.mss
+++ b/placenames.mss
@@ -6,11 +6,13 @@
 .country {
   [admin_level = '2'][zoom >= 3][way_pixels > 1000][way_pixels < 360000] {
     text-name: "[name]";
-    text-size: 9;
+    text-size: 10;
 
+    /*
     [zoom >= 3] {
       text-size: 10;
     }
+    */
     [zoom >= 4] {
       text-size: 11;
     }
@@ -70,8 +72,8 @@
       text-name: "[name]";
       text-size: 11;
       text-fill: @placenames;
-      text-face-name: @book-fonts;
-      text-halo-radius: 1.5;
+      text-face-name: @bold-fonts;
+      text-halo-radius: 2.5;
       text-halo-fill: rgba(255,255,255,0.6);
       text-wrap-width: 30;
       text-min-distance: 10;
@@ -106,8 +108,8 @@
       text-name: "[name]";
       text-size: 10;
       text-fill: @placenames;
-      text-face-name: @book-fonts;
-      text-halo-radius: 1.5;
+      text-face-name: @bold-fonts;
+      text-halo-radius: 2;
       text-halo-fill: rgba(255,255,255,0.6);
       text-wrap-width: 30;
       text-min-distance: 10;
@@ -116,12 +118,13 @@
         text-wrap-width: 60;
       }
       [zoom >= 11] {
-        text-size: 14;
+        text-size: 15;
         text-wrap-width: 70;
       }
       [zoom >= 14] {
-        text-size: 15;
+        // text-size: 15;
         text-wrap-width: 75;
+        text-fill: @placenames-light;
       }
     }
   }
@@ -134,7 +137,7 @@
       text-size: 10;
       text-fill: @placenames;
       text-face-name: @book-fonts;
-      text-halo-radius: 1.5;
+      text-halo-radius: 2;
       text-halo-fill: rgba(255,255,255,0.6);
       text-wrap-width: 45;
       text-min-distance: 10;
@@ -149,6 +152,7 @@
       [zoom >= 14] {
         text-size: 15;
         text-wrap-width: 70;
+        text-fill: @placenames-light;
       }
     }
   }
@@ -179,7 +183,7 @@
 
 #placenames-small::village {
   [place = 'village'] {
-    [zoom >= 12][zoom < 17] {
+    [zoom >= 10][zoom < 17] {
       text-name: "[name]";
       text-size: 10;
       text-fill: @placenames;
