--- a/placenames.mss
+++ b/placenames.mss
@@ -172,7 +172,7 @@
     }
     [zoom >= 8][score >= 400000] {
       text-name: '[name]';
-      text-face-name: @book-fonts;
+      text-face-name: @bold-fonts;
       text-fill: @placenames;
       text-size: 13;
       text-wrap-width: 65; // 5.0 em
@@ -207,7 +207,7 @@
       shield-name: "[name]";
       shield-size: 10;
       shield-fill: @placenames;
-      shield-face-name: @book-fonts;
+      shield-face-name: @bold-fonts;
       shield-halo-fill: @standard-halo-fill;
       shield-halo-radius: @standard-halo-radius * 1.5;
       shield-wrap-width: 30; // 3.0 em
@@ -331,7 +331,7 @@
 
 #placenames-small::village {
   [place = 'village'] {
-    [zoom >= 12][zoom < 17] {
+    [zoom >= 10][zoom < 17] {
       text-name: "[name]";
       text-size: 10;
       text-fill: @placenames;
