--- a/placenames.mss
+++ b/placenames.mss
@@ -8,11 +8,13 @@
 .country {
   [admin_level = '2'][zoom >= 3][way_pixels > 1000][way_pixels < 360000] {
     text-name: "[name]";
-    text-size: 9;
+    text-size: 10;
 
+    /*
     [zoom >= 3] {
       text-size: 10;
     }
+    */
     [zoom >= 4] {
       text-size: 11;
     }
@@ -70,15 +72,15 @@
     [zoom >= 4][score >= 3000000],
     [zoom >= 5][score >= 400000] {
       text-name: "[name]";
-      text-size: 8;
+      text-size: 10;
       text-fill: @placenames;
-      text-face-name: @book-fonts;
-      text-halo-radius: 1.5;
+      text-face-name: @bold-fonts;
+      text-halo-radius: 2.5;
       text-halo-fill: rgba(255,255,255,0.6);
       text-wrap-width: 30;
       text-min-distance: 10;
       [zoom >= 5] {
-        text-size: 10;
+        // text-size: 10;
         text-wrap-width: 45;
       }
       [zoom >= 6] {
@@ -98,24 +100,25 @@
     [zoom >= 6][score >= 70000],
     [zoom >= 7] {
       text-name: "[name]";
-      text-size: 9;
+      text-size: 10;
       text-fill: @placenames;
-      text-face-name: @book-fonts;
-      text-halo-radius: 1.5;
+      text-face-name: @bold-fonts;
+      text-halo-radius: 2;
       text-halo-fill: rgba(255,255,255,0.6);
-      text-wrap-width: 30;
+      text-wrap-width: 45;
       text-min-distance: 10;
       [zoom >= 9] {
-        text-size: 11;
+        text-size: 12;
         text-wrap-width: 60;
       }
       [zoom >= 11] {
-        text-size: 14;
+        text-size: 15;
         text-wrap-width: 70;
       }
       [zoom >= 14] {
-        text-size: 15;
+        // text-size: 15;
         text-wrap-width: 75;
+        text-fill: @placenames-light;
       }
     }
   }
@@ -125,10 +128,10 @@
   [category = 2] {
     [zoom >= 9][zoom < 16] {
       text-name: "[name]";
-      text-size: 9;
+      text-size: 10;
       text-fill: @placenames;
       text-face-name: @book-fonts;
-      text-halo-radius: 1.5;
+      text-halo-radius: 2;
       text-halo-fill: rgba(255,255,255,0.6);
       text-wrap-width: 45;
       text-min-distance: 10;
@@ -143,6 +146,7 @@
       [zoom >= 14] {
         text-size: 15;
         text-wrap-width: 70;
+        text-fill: @placenames-light;
       }
     }
   }
@@ -173,7 +177,7 @@
 
 #placenames-small::village {
   [place = 'village'] {
-    [zoom >= 12][zoom < 17] {
+    [zoom >= 10][zoom < 17] {
       text-name: "[name]";
       text-size: 10;
       text-fill: @placenames;
