--- a/placenames.mss
+++ b/placenames.mss
@@ -63,8 +63,8 @@
       text-name: "[name]";
       text-size: 11;
       text-fill: @placenames;
-      text-face-name: @book-fonts;
-      text-halo-radius: 1.5;
+      text-face-name: @bold-fonts;
+      text-halo-radius: 2.5;
       text-halo-fill: rgba(255,255,255,0.6);
       text-wrap-width: 30;
       [zoom >= 5] {
@@ -98,8 +98,8 @@
       text-name: "[name]";
       text-size: 10;
       text-fill: @placenames;
-      text-face-name: @book-fonts;
-      text-halo-radius: 1.5;
+      text-face-name: @bold-fonts;
+      text-halo-radius: 2;
       text-halo-fill: rgba(255,255,255,0.6);
       text-wrap-width: 30;
       [zoom >= 9] {
@@ -107,12 +107,13 @@
         text-wrap-width: 60;
       }
       [zoom >= 11] {
-        text-size: 14;
+        text-size: 15;
         text-wrap-width: 70;
       }
       [zoom >= 14] {
-        text-size: 15;
+        // text-size: 15;
         text-wrap-width: 75;
+        text-fill: @placenames-light;
       }
     }
   }
@@ -125,7 +126,7 @@
       text-size: 10;
       text-fill: @placenames;
       text-face-name: @book-fonts;
-      text-halo-radius: 1.5;
+      text-halo-radius: 2;
       text-halo-fill: rgba(255,255,255,0.6);
       text-wrap-width: 45;
       [zoom >= 11] {
@@ -139,6 +140,7 @@
       [zoom >= 14] {
         text-size: 15;
         text-wrap-width: 70;
+        text-fill: @placenames-light;
       }
     }
   }
@@ -168,7 +170,7 @@
 
 #placenames-small::village {
   [place = 'village'] {
-    [zoom >= 12][zoom < 17] {
+    [zoom >= 10][zoom < 17] {
       text-name: "[name]";
       text-size: 10;
       text-fill: @placenames;
