--- a/INSTALL.md
+++ b/INSTALL.md
@@ -34,7 +34,9 @@
 You can also download them manually at the following paths:
 
 * [`simplified-land-polygons.shp`](http://data.openstreetmapdata.com/simplified-land-polygons-complete-3857.zip) (updated daily)
+* [`simplified-water-polygons.shp`](http://data.openstreetmapdata.com/simplified-water-polygons-complete-3857.zip) (updated daily)
 * [`land-polygon.shp`](http://data.openstreetmapdata.com/land-polygons-split-3857.zip) (updated daily)
+* [`water-polygon.shp`](http://data.openstreetmapdata.com/water-polygons-split-3857.zip) (updated daily)
 * [`builtup_area.shp`](http://planet.openstreetmap.org/historical-shapefiles/world_boundaries-spherical.tgz)
 * [`ne_110m_admin_0_boundary_lines_land.shp`](http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/110m/cultural/ne_110m_admin_0_boundary_lines_land.zip)
 
--- a/project.mml
+++ b/project.mml
@@ -53,24 +53,6 @@
   - admin.mss
   - addressing.mss
 Layer:
-  - id: world
-    name: world
-    geometry: polygon
-    <<: *extents
-    Datasource:
-      file: data/simplified-land-polygons-complete-3857/simplified_land_polygons.shp
-      type: shape
-    properties:
-      maxzoom: 9
-  - id: coast-poly
-    name: coast-poly
-    geometry: polygon
-    <<: *extents
-    Datasource:
-      file: data/land-polygons-split-3857/land_polygons.shp
-      type: shape
-    properties:
-      minzoom: 10
   - id: builtup
     name: builtup
     geometry: polygon
@@ -216,6 +198,26 @@
     properties:
       minzoom: 8
       maxzoom: 11
+  - id: ocean-lz
+    name: ocean-lz
+    class: ocean
+    geometry: polygon
+    <<: *extents
+    Datasource:
+      file: data/simplified-water-polygons-complete-3857/simplified_water_polygons.shp
+      type: shape
+    properties:
+      maxzoom: 9
+  - id: ocean
+    name: ocean
+    class: ocean
+    geometry: polygon
+    <<: *extents
+    Datasource:
+      file: data/water-polygons-split-3857/water_polygons.shp
+      type: shape
+    properties:
+      minzoom: 10
   - id: icesheet-poly
     name: icesheet-poly
     geometry: polygon
--- a/water.mss
+++ b/water.mss
@@ -25,6 +25,7 @@
     }
   }
 
+  /*
   [waterway = 'dock'],
   [waterway = 'canal'] {
     [zoom >= 9]::waterway {
@@ -61,6 +62,30 @@
       }
     }
   }
+  */
+}
+
+/*
+ * Water areas, of all types. Because they are rendered all the same,
+ * attachments can be used to reduce combinational rules
+ */
+
+#water-areas[waterway = 'dock'][zoom >= 9]::waterway,
+#water-areas[waterway = 'canal'][zoom >= 9]::waterway,
+#water-areas[landuse = 'basin'][zoom >= 7]::landuse,
+#water-areas[natural = 'water'][zoom >= 6]::natural,
+#water-areas[landuse = 'reservoir'][zoom >= 6]::landuse,
+#water-areas[waterway = 'riverbank'][zoom >= 6]::waterway,
+.ocean {
+  polygon-fill: @water-color;
+
+  // Only the SQL layers have way_pixels
+  #water-areas[way_pixels >= 4] {
+    polygon-gamma: 0.75;
+  }
+  #water-areas[way_pixels >= 64] {
+    polygon-gamma: 0.6;
+  }
 }
 
 #water-lines-casing {
--- a/scripts/get-shapefiles.py
+++ b/scripts/get-shapefiles.py
@@ -82,7 +82,24 @@
         'type': 'zip',
         'index': ['icesheet_outlines.shp'],
         'long_opt': '--icesheet-outlines'
-    }
+    },
+
+    7: {
+        'directory': 'simplified-water-polygons-complete-3857',
+        'url': 'http://data.openstreetmapdata.com/simplified-water-polygons-complete-3857.zip',  # noqa
+        'type': 'zip',
+        'index': ['simplified_water_polygons.shp'],
+        'long_opt': '--simplified-water'
+    },
+
+
+    8: {
+        'directory': 'water-polygons-split-3857',
+        'url': 'http://data.openstreetmapdata.com/water-polygons-split-3857.zip',  # noqa
+        'type': 'zip',
+        'index': ['water_polygons.shp'],
+        'long_opt': '--water-polygons'
+    },
 }
 
 u_prompt = True
