--- a/landcover.mss
+++ b/landcover.mss
@@ -8,6 +8,18 @@
 @park: #c8facc;         // Lch(94,30,145) also recreation_ground
 @orchard: #aedfa3;
 
+// --- Reliefs
+@ridge-text:          #7A2F18;
+@valley-text:         #305040;
+@mountain_range-text: #C03A01;
+@mountain_area-text:  @mountain_range-text;
+@massif-text:         #C03A01;
+@cliff-text:          #999;
+@arete-text:          #800000;
+@couloir-text:        #0080C0;
+@dale-text:           #008000;
+@gorge-text:          #800040;
+
 // --- sports ---
 
 @stadium: #3c9; // also sports_centre
@@ -670,7 +682,33 @@
   }
 }
 
-#cliffs {
+#reliefsymbols {
+  [natural = 'arete'][zoom >= 12] {
+    line-pattern-smooth: 0.3;
+    /*
+    line-pattern-file: url('symbols/arete_small.png');
+    [zoom >= 15] {
+      line-pattern-file: url('symbols/arete_medium.png');
+    }
+    [zoom >= 16] {
+      line-pattern-file: url('symbols/arete_large.png');
+    }
+    */
+    line-pattern-file: url('symbols/arete.svg');
+  }
+  [natural = 'ridge'][zoom >= 12] {
+    /*
+    line-pattern-smooth: 0.3;
+    line-pattern-file: url('symbols/ridge_small.png');
+    [zoom >= 15] {
+      line-pattern-file: url('symbols/ridge_medium.png');
+    }
+    [zoom >= 16] {
+      line-pattern-file: url('symbols/ridge_large.png');
+    }
+    */
+    line-pattern-file: url('symbols/ridge.svg');
+  }
   [natural = 'cliff'][zoom >= 13] {
     // line-pattern-file: url('symbols/cliff.png');
     line-pattern-file: url('symbols/cliff_mdione.png');
@@ -736,18 +774,214 @@
   }
 }
 
+#text-point[zoom >= 15],
 #text-line {
-  [feature = 'natural_cliff'][zoom >= 15],
+  [feature = 'natural_cliff'][zoom >= 11],
   [feature = 'man_made_embankment'][zoom >= 15] {
     text-name: "[name]";
     text-halo-radius: 1;
     text-halo-fill: rgba(255,255,255,0.6);
-    text-fill: #999;
+    text-fill: @cliff-text;
     text-size: 10;
     text-face-name: @book-fonts;
-    text-placement: line;
     text-dy: 8;
     text-vertical-alignment: middle;
     text-spacing: 400;
+    #text-line {
+      text-placement: line;
+    }
+    #text-point {
+      text-placement: point;
+      text-dy: 8;
+      text-size: 10;
+    }
+  }
+  [feature = 'natural_dale'][zoom >= 13],
+  [feature = 'natural_gorge'][zoom >= 14],
+  [feature = 'natural_couloir'][zoom >= 14],
+  [feature = 'natural_ridge'][zoom >= 8]::foreground,
+  [feature = 'natural_arete'][zoom >= 8]::foreground,
+  [feature = 'natural_massif'][zoom >= 15]#text-point, // point visibility given to mappers at high zoom, to help debugging tags to be changed to polylines
+  [feature = 'natural_mountain_range'][zoom >= 15]#text-point, // point visibility given to mappers at high zoom, to help debugging tags to be changed to polylines
+  [feature = 'place_mountain_area'][zoom >= 15]#text-point, // point visibility given to mappers at high zoom, to help debugging tags to be changed to polylines
+  [feature = 'natural_valley'][zoom >= 8]::foreground {
+    [feature = 'natural_dale'] { text-fill: @dale-text; }
+    [feature = 'natural_gorge'] { text-fill: @gorge-text; }
+    [feature = 'natural_couloir'] { text-fill: @couloir-text; }
+    [feature = 'natural_ridge'] { text-fill: @ridge-text; }
+    [feature = 'natural_arete']  { text-fill: @arete-text; }
+    [feature = 'natural_valley'] { text-fill: @valley-text; }
+    [feature = 'natural_massif'] {
+      text-fill: @massif-text;
+      text-transform: uppercase;
+    }
+    [feature = 'natural_mountain_range'] {
+      text-fill: @mountain_range-text;
+      text-transform: uppercase;
+    }
+    [feature = 'place_mountain_area'] {
+      text-fill: @mountain_area-text;
+      text-transform: uppercase;
+    }
+    // general settings
+    text-halo-radius: 2;
+    text-halo-fill: rgba(255,255,255,0.6);
+    text-opacity: 1;
+    [zoom >= 10] { text-opacity: 0.7; }
+    text-dx: 8;
+    text-dy: 8;
+    text-spacing: 760;
+    text-face-name: @oblique-fonts;
+    text-wrap-width: 2;
+    text-name: "[name]";
+    // the following text-line and spacing dimensions are independent from pixel_length (which needs text-line)
+    [zoom >= 10] { text-size: 10; text-opacity: 0.5; }
+    [zoom >= 11] { text-size: 11; text-character-spacing: 4; text-opacity: 0.6; }
+    [zoom >= 12] { text-size: 13; text-character-spacing: 4; text-opacity: 0.6; }
+    [zoom >= 13] { text-size: 13; text-character-spacing: 5; text-opacity: 0.6; }
+    [zoom >= 14] { text-size: 12; text-character-spacing: 3; }
+    [zoom >= 15] {
+      text-size: 16;
+      text-dx: 15;
+      text-dy: 15;
+      text-character-spacing: 6;
+    }
+    [zoom >= 16] { text-size: 18; text-character-spacing: 12; }
+    #text-line {
+      text-max-char-angle-delta: 30;
+      text-placement: line;
+      text-placement-type: simple;
+      [zoom < 15] {
+        // text-placements: "N,S,E,W,NE,SE,NW,SW,18,16,14,12,10,9,8";
+      }
+      // the following text-line and spacing dimensions depend on pixel_length (e.g., differentiate long lines)
+      [pixel_length > 60] {
+        text-dx: 11;
+        text-dy: 11;
+        text-size: 11;
+        text-character-spacing: 3;
+        text-wrap-width: 100;
+      }
+      [zoom = 10][pixel_length > 125] {
+        text-character-spacing: 2;
+        text-size: 13;
+      }
+      [zoom = 11][pixel_length > 250] {
+        text-size: 13;
+        text-character-spacing: 6;
+        text-opacity: 0.6;
+      }
+      [zoom = 12][pixel_length > 500] {
+        text-size: 14;
+        text-character-spacing: 7;
+        text-opacity: 0.6;
+        text-transform: uppercase;
+      }
+      [zoom = 13][pixel_length > 1000] {
+        text-size: 15;
+        text-character-spacing: 10;
+        text-opacity: 0.6;
+        text-max-char-angle-delta: 10;
+        text-transform: uppercase;
+      }
+      [zoom = 14][pixel_length > 2000] {
+        text-size: 20;
+        text-character-spacing: 10;
+        text-opacity: 0.5;
+        text-max-char-angle-delta: 10;
+        text-transform: uppercase;
+        text-placement-type: dummy;
+      }
+      [zoom >= 15][pixel_length > 4000] {
+        text-size: 25;
+        text-character-spacing: 20;
+        text-opacity: 0.5;
+        text-max-char-angle-delta: 10;
+        text-transform: uppercase;
+        text-placement-type: dummy;
+      }
+      [zoom < 16][name=~'^.{14,}$'] { // improve probability for long names (>=14 characters) to be shown
+        text-max-char-angle-delta: 30;
+        text-transform: none;
+      }
+    }
+    [feature = 'natural_dale'],
+    [feature = 'natural_gorge'],
+    [feature = 'natural_couloir'],
+    [feature = 'natural_arete'],
+    [feature = 'natural_ridge'] {
+      [zoom >= 10] { text-size: 10; }
+      [zoom >= 11] { text-size: 12; }
+      [zoom >= 12] { text-size: 12; }
+      [zoom >= 13] { text-size: 12; }
+      [zoom >= 14] { text-size: 12; }
+      [zoom >= 15] { text-size: 15; }
+      [zoom >= 16] { text-size: 18; }
+    }
+    #text-point { // points are only shown for debugging purpose, with fixed rendering
+      text-placement: point;
+      text-character-spacing: 0;
+      text-size: 10;
+    }
+  }
+}
+
+#landcover-low-zoom[zoom < 10]::foreground,
+#amenity-points-poly[zoom >= 10] {
+  [feature = 'natural_dale'][zoom >= 14],
+  [feature = 'natural_gorge'][zoom >= 15],
+  [feature = 'natural_couloir'][zoom >= 15],
+  [feature = 'natural_arete'][zoom >= 15],
+  [feature = 'natural_ridge'][zoom >= 15],
+  [feature = 'natural_massif'][zoom >= 8],
+  [feature = 'natural_mountain_range'][zoom >= 8][zoom < 13],
+  [feature = 'place_mountain_area'][zoom >= 8][zoom < 13],
+  [feature = 'natural_valley'] {
+    text-halo-radius: 2.5;
+    text-halo-fill: rgba(255,255,255,0.6);
+    text-vertical-alignment: middle;
+    text-opacity: 1;
+    text-spacing: 760;
+    text-face-name: @oblique-fonts;
+    text-wrap-width: 2;
+    text-placement: interior;
+    text-placement-type: simple;
+    text-size: 8;
+    [zoom >= 9] { text-size: 9; }
+    [zoom < 11] {
+      // text-placements: "N,S,E,W,NE,SE,NW,SW,10,9,8,7";
+      text-placements: "N,S,E,W";
+    }
+    [zoom >= 10] {
+      text-size: 10;
+      text-opacity: 0.7;
+      [way_pixels > 4000] { text-size: 12; }
+      [way_pixels > 20000][zoom >= 10] { text-size: 14; }
+      [way_pixels > 60000][zoom >= 11] { text-size: 17; }
+      [way_pixels > 200000][zoom >= 12] { text-size: 21; }
+      [way_pixels > 500000][zoom >= 13] { text-size: 25; }
+    }
+    text-name: "[name]";
+    [feature = 'natural_dale'] { text-fill: @dale-text; }
+    [feature = 'natural_gorge'] { text-fill: @gorge-text; }
+    [feature = 'natural_couloir'] { text-fill: @couloir-text; }
+    [feature = 'natural_ridge'] { text-fill: @ridge-text; }
+    [feature = 'natural_arete']  { text-fill: @arete-text; }
+    [feature = 'natural_massif'] {
+      text-fill: @massif-text;
+      text-transform: uppercase;
+      [zoom>=9] { text-character-spacing: 3; }
+    }
+    [feature = 'natural_mountain_range'] {
+      text-fill: @mountain_range-text;
+      text-transform: uppercase;
+      [zoom>=9] { text-character-spacing: 3; }
+    }
+    [feature = 'place_mountain_area'] {
+      text-fill: @mountain_area-text;
+      text-transform: uppercase;
+      [zoom>=9] { text-character-spacing: 3; }
+    }
+    [feature = 'natural_valley'] { text-fill: @valley-text; }
   }
 }
--- a/project.yaml
+++ b/project.yaml
@@ -879,8 +879,8 @@
     properties:
       minzoom: 14
     advanced: {}
-  - id: "cliffs"
-    name: "cliffs"
+  - id: "reliefsymbols"
+    name: "reliefsymbols"
     class: ""
     geometry: "linestring"
     <<: *extents
@@ -888,12 +888,12 @@
       <<: *osm2pgsql
       table: |-
         (SELECT
-            way, "natural", man_made
+            way, "natural", name, ST_Length(way)/NULLIF(SQRT(!pixel_width!::real*!pixel_width!::real),0) AS pixel_length, man_made
           FROM planet_osm_line
-          WHERE "natural" = 'cliff' OR man_made = 'embankment'
-        ) AS cliffs
+          WHERE "natural" in ('cliff', 'arete', 'ridge') OR man_made = 'embankment'
+        ) AS reliefsymbols
     properties:
-      minzoom: 13
+      minzoom: 10
     advanced: {}
   - id: "area-barriers"
     name: "area-barriers"
@@ -1768,7 +1768,7 @@
                                             'video_games', 'watches', 'wholesale', 'yes') THEN 'other' ELSE NULL END,
               'leisure_' || CASE WHEN leisure IN ('water_park', 'playground', 'miniature_golf', 'golf_course', 'picnic_table') THEN leisure ELSE NULL END,
               'man_made_' || CASE WHEN man_made IN ('mast', 'water_tower', 'lighthouse', 'windmill') THEN man_made ELSE NULL END,
-              'natural_' || CASE WHEN "natural" IN ('spring') THEN "natural" ELSE NULL END,
+              'natural_' || CASE WHEN "natural" IN ('spring', 'valley', 'ridge', 'cliff', 'arete', 'dale', 'gorge', 'couloir', 'mountain_range', 'massif') THEN "natural" ELSE NULL END,
               'historic_' || CASE WHEN historic IN ('memorial', 'monument', 'archaeological_site', 'castle', 'fort') THEN historic ELSE NULL END,
               'highway_'|| CASE WHEN highway IN ('bus_stop', 'elevator', 'traffic_signals') THEN highway ELSE NULL END,
               'power_' || CASE WHEN power IN ('generator') THEN power ELSE NULL END,
@@ -1779,6 +1779,7 @@
             denomination,
             "generator:source",
             power_source,
+            name,
             castle_type,
             way_area/NULLIF(!pixel_width!::real*!pixel_height!::real,0) AS way_pixels
           FROM planet_osm_polygon
@@ -1797,7 +1798,7 @@
             OR shop IS NOT NULL -- skip checking a huge list and use a null check
             OR leisure IN ('water_park', 'playground', 'miniature_golf', 'golf_course', 'picnic_table')
             OR man_made IN ('mast', 'water_tower', 'lighthouse', 'windmill')
-            OR "natural" IN ('spring')
+            OR "natural" IN ('spring', 'valley', 'ridge', 'cliff', 'arete', 'dale', 'gorge', 'couloir', 'mountain_range', 'massif')
             OR historic IN ('memorial', 'monument', 'archaeological_site', 'castle', 'fort')
             OR highway IN ('bus_stop', 'elevator', 'traffic_signals')
             OR (power = 'generator' AND ("generator:source" = 'wind' OR power_source = 'wind'))
@@ -2199,12 +2200,13 @@
             name,
             operator,
             ref,
+            ST_Length(way)/NULLIF(SQRT(!pixel_width!::real*!pixel_width!::real),0) AS pixel_length,
             NULL AS way_area,
             CASE WHEN building = 'no' OR building IS NULL THEN 'no' ELSE 'yes' END AS is_building
           FROM planet_osm_line
           WHERE (man_made IN ('pier', 'breakwater', 'groyne', 'embankment')
               OR waterway IN ('dam', 'weir')
-              OR "natural" IN ('cliff'))
+              OR "natural" IN ('cliff', 'ridge', 'valley', 'glacier', 'arete', 'dale', 'gorge', 'couloir', 'mountain_range', 'massif'))
             AND name IS NOT NULL
         ) AS text_line
     advanced: {}
@@ -2285,9 +2287,11 @@
                                                       'military') THEN landuse ELSE NULL END,
                   'man_made_' || CASE WHEN man_made IN ('lighthouse', 'windmill', 'mast', 'water_tower') THEN man_made ELSE NULL END,
                   'natural_' || CASE WHEN "natural" IN ('wood', 'peak', 'volcano', 'saddle', 'cave_entrance', 'water', 'mud', 'wetland', 'marsh', 'bay', 'spring', 
-                                                        'scree', 'shingle', 'bare_rock', 'sand', 'heath', 'grassland', 'scrub', 'beach', 'glacier', 'tree') 
+                                                        'scree', 'shingle', 'bare_rock', 'sand', 'heath', 'grassland', 'scrub', 'beach', 'cliff', 'valley',
+                                                        'ridge', 'arete', 'dale', 'gorge', 'couloir', 'mountain_range', 'massif', 'glacier', 'tree')
                                                         THEN "natural" ELSE NULL END,
-                  'place_' || CASE WHEN place IN ('island', 'islet') THEN place ELSE NULL END,
+                  'place_' || CASE WHEN place IN ('island', 'islet') THEN place
+                                  ELSE NULL END,
                   'military_' || CASE WHEN military IN ('danger_area') THEN military ELSE NULL END,
                   'historic_' || CASE WHEN historic IN ('memorial', 'monument', 'archaeological_site', 'castle', 'fort') THEN historic ELSE NULL END,
                   'highway_' || CASE WHEN highway IN ('services', 'rest_area', 'bus_stop', 'elevator', 'ford') THEN highway ELSE NULL END,
