--- a/roads.mss
+++ b/roads.mss
@@ -2769,25 +2769,31 @@
   [highway = 'primary'] {
     [zoom >= 13] {
       text-name: "[name]";
-      text-size: 8;
+      text-size: 10;
       text-fill: black;
       text-spacing: 300;
       text-clip: false;
       text-placement: line;
       text-face-name: @book-fonts;
       [tunnel = 'no'] {
+        /*
         text-halo-radius: 1;
         [highway = 'motorway'] { text-halo-fill: @motorway-fill; }
         [highway = 'trunk'] { text-halo-fill: @trunk-fill; }
         [highway = 'primary'] { text-halo-fill: @primary-fill; }
+        */
+        text-halo-radius: 1.5;
+        text-halo-fill: white;
       }
     }
+    /*
     [zoom >= 14] {
       text-size: 9;
     }
     [zoom >= 15] {
       text-size: 10;
     }
+    */
     [zoom >= 17] {
       text-size: 11;
     }
@@ -2798,21 +2804,23 @@
   [highway = 'secondary'] {
     [zoom >= 13] {
       text-name: "[name]";
-      text-size: 8;
+      text-size: 10;
       text-fill: black;
       text-spacing: 300;
       text-clip: false;
       text-placement: line;
       text-face-name: @book-fonts;
-      text-halo-radius: 1;
-      text-halo-fill: @secondary-fill;
+      text-halo-radius: 1.5;
+      text-halo-fill: white;
     }
+    /*
     [zoom >= 14] {
       text-size: 9;
     }
     [zoom >= 15] {
       text-size: 10;
     }
+    */
     [zoom >= 17] {
       text-size: 11;
     }
@@ -2824,14 +2832,14 @@
   [highway = 'tertiary_link'] {
     [zoom >= 14] {
       text-name: "[name]";
-      text-size: 9;
+      text-size: 10;
       text-fill: black;
       text-spacing: 300;
       text-clip: false;
       text-placement: line;
       text-face-name: @book-fonts;
-      text-halo-radius: 1;
-      text-halo-fill: @tertiary-fill;
+      text-halo-radius: 1.5;
+      text-halo-fill: white;
     }
     [zoom >= 17] {
       text-size: 11;
@@ -2867,18 +2875,20 @@
   [highway = 'road'] {
     [zoom >= 15] {
       text-name: "[name]";
-      text-size: 8;
+      text-size: 10;
       text-fill: black;
       text-spacing: 300;
       text-clip: false;
       text-placement: line;
       text-halo-radius: 1;
-      text-halo-fill: @residential-fill;
+      text-halo-fill: white;
       text-face-name: @book-fonts;
     }
+    /*
     [zoom >= 16] {
       text-size: 9;
     }
+    */
     [zoom >= 17] {
       text-size: 11;
       text-spacing: 400;
