--- a/style.mss
+++ b/style.mss
@@ -16,8 +16,99 @@
                 "Droid Sans Fallback Regular", "Unifont Medium";
 
 @water-color: #b5d0d0;
+@water-dark: #5A5AA1;
+@water-text: #5A5AA1;
 @land-color: #bfbfbf;
 
+// admin
+@admin-boundaries: purple;
+
+// amenities
+@attraction: #f2caea;
+@barracks: #ff8f8f;
+
+@darken-lighten: 15%;
+
+// roads
+@oneway: #aaa;
+
+@motorway-casing: darken(@motorway-fill, @darken-lighten);
+@motorway-low-zoom: @motorway-fill;
+@motorway-fill: #cf3030;
+@motorway-tunnel-fill: lighten(@motorway-fill, @darken-lighten);
+
+@trunk-casing: darken(@trunk-fill, @darken-lighten);
+@trunk-low-zoom: #cf6868;
+@trunk-fill: @motorway-fill;
+@trunk-tunnel-fill: lighten(@trunk-fill, @darken-lighten);
+
+@primary-casing: darken(@primary-fill, @darken-lighten);
+@primary-low-zoom: @primary-fill;
+@primary-fill: #8a5c00;
+@primary-tunnel-fill: lighten(@primary-fill, @darken-lighten);
+
+@secondary-casing: darken(@secondary-fill, @darken-lighten);
+@secondary-low-zoom: @secondary-fill;
+@secondary-fill: #2f6f2f;
+@secondary-tunnel-fill: lighten(@secondary-fill, @darken-lighten);
+
+@tertiary-casing: #444;
+@tertiary-fill: #ffffff;
+@tertiary-tunnel-fill: lighten(@tertiary-fill, 5%);
+
+@residential-casing: #444;
+@residential-fill: #ffffff;
+@residential-tunnel-fill: lighten(@residential-fill, 10%);
+
+@living-street-casing: @default-casing;
+@living-street-fill: #ededed;
+
+/*
+minor roads for different zoom levels
+tertiary    10-12
+residential 12-13
+road        10-
+service     13
+*/
+@unimportant-road: #bbb;
+
+@service-casing: #999;
+@service-fill: @residential-fill;
+
+@pedestrian-casing: grey;
+@pedestrian-fill: #dddde8;
+
+@road-casing: @residential-casing;
+@road-fill: #ddd;
+
+@path-casing: @default-casing;
+@path-fill: black;
+
+@footway-casing: @default-casing;
+@footway-fill: salmon;
+
+@steps-casing: @default-casing;
+@steps-fill: @footway-fill;
+
+@cycleway-casing: @default-casing;
+@cycleway-fill: blue;
+
+@bridleway-casing: @default-casing;
+@bridleway-fill: green;
+
+@byway-casing: @default-casing;
+@byway-fill: #ffcc00;
+
+@track-casing: @default-casing;
+@track-fill: #996600;
+@track-grade1-fill: #b37700;
+@track-grade2-fill: #a87000;
+
+@aeroway-fill: #bbc;
+@runway-fill: @aeroway-fill;
+@taxiway-fill: @aeroway-fill;
+@helipad-fill: @aeroway-fill;
+
 // terrain-small [0-6], terrain-medium [7-8], terrain-big [9-]
 .terrain {
   raster-scaling: lanczos;
--- a/roads.mss
+++ b/roads.mss
@@ -1286,11 +1286,11 @@
 
     [feature = 'highway_secondary'] {
       [zoom >= 9] {
-        line-color: @unimportant-road;
+        line-color: @secondary-low-zoom;
         line-width: @secondary-width-z9;
       }
       [zoom >=10] {
-        line-color: @unimportant-road;
+        line-color: @secondary-low-zoom;
         line-width: @secondary-width-z10;
       }
       [zoom >= 11] { line-width: @secondary-width-z11; }
@@ -2065,6 +2065,23 @@
       }
     }
   }
+
+  ::strip {
+    [feature = 'highway_motorway'][link = 'no'],
+    [feature = 'highway_trunk'][link = 'no'] {
+      [zoom >= 12] {
+        line-width: 1;
+        [feature = 'highway_motorway'] {
+          line-color: yellow;
+        }
+        [feature = 'highway_trunk'][link = 'no'] {
+          line-color: white;
+        }
+      }
+      [zoom >= 14] { line-width: 1.5; }
+      [zoom >= 16] { line-width: 2; }
+    }
+  }
 }
 
 #turning-circle-casing {
