--- a/project.mml
+++ b/project.mml
@@ -53,6 +53,40 @@
   - admin.mss
   - addressing.mss
 Layer:
+  - id: "terrain-small"
+    name: "terrain-small"
+    class: "terrain"
+    geometry: "raster"
+    <<: *extents
+    Datasource:
+      file: "data/height/mixed-terrain-small.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      maxzoom: 6
+  - id: "terrain-medium"
+    name: "terrain-medium"
+    class: "terrain"
+    geometry: "raster"
+    <<: *extents
+    Datasource:
+      file: "data/height/mixed-terrain-medium.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 7
+      maxzoom: 8
+  - id: "terrain-big"
+    name: "terrain-big"
+    class: "terrain"
+    geometry: "raster"
+    <<: *extents
+    Datasource:
+      file: "data/height/mixed-terrain.vrt"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 8
   - id: builtup
     name: builtup
     geometry: polygon
@@ -166,6 +200,100 @@
         ) AS landcover_line
     properties:
       minzoom: 14
+  - id: "hillshade-small"
+    name: "hillshade-small"
+    class: "hillshade-over"
+    geometry: "raster"
+    <<: *extents
+    Datasource:
+      file: "data/height/mixed-hillshade-small.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      maxzoom: 6
+  - id: "hillshade-medium-over"
+    name: "hillshade-medium-over"
+    class: "hillshade-over"
+    geometry: "raster"
+    <<: *extents
+    Datasource:
+      file: "data/height/mixed-hillshade-medium.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 7
+      maxzoom: 8
+  - id: "hillshade-medium-overlay"
+    name: "hillshade-medium-overlay"
+    class: "hillshade-overlay"
+    geometry: "raster"
+    <<: *extents
+    Datasource:
+      file: "data/height/mixed-hillshade-medium.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 8
+      maxzoom: 8
+  - id: "hillshade-big"
+    name: "hillshade-big"
+    class: "hillshade-overlay"
+    geometry: "raster"
+    <<: *extents
+    Datasource:
+      file: "data/height/mixed-hillshade.vrt"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 9
+  - id: "slopeshade-small"
+    name: "slopeshade-small"
+    class: "slopeshade-over"
+    geometry: "raster"
+    <<: *extents
+    Datasource:
+      file: "data/height/mixed-slopeshade-small.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      maxzoom: 6
+    # status: off
+  - id: "slopeshade-medium-over"
+    name: "slopeshade-medium-over"
+    class: "slopeshade-over"
+    geometry: "raster"
+    <<: *extents
+    Datasource:
+      file: "data/height/mixed-slopeshade-medium.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 7
+      maxzoom: 7
+    # status: off
+  - id: "slopeshade-medium-overlay"
+    name: "slopeshade-medium-overlay"
+    class: "slopeshade-overlay"
+    geometry: "raster"
+    <<: *extents
+    Datasource:
+      file: "data/height/mixed-slopeshade-medium.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 8
+      maxzoom: 8
+  - id: "slopeshade-big"
+    name: "slopeshade-big"
+    class: "slopeshade-overlay"
+    geometry: "raster"
+    <<: *extents
+    Datasource:
+      file: "data/height/mixed-slopeshade.vrt"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 9
   - id: water-lines-casing
     name: water-lines-casing
     geometry: linestring
