--- a/project.mml
+++ b/project.mml
@@ -53,6 +53,40 @@
   - admin.mss
   - addressing.mss
 Layer:
+  - id: "terrain-small"
+    name: "terrain-small"
+    class: "terrain"
+    geometry: "raster"
+    <<: *extents84
+    Datasource:
+      file: "data/height/mixed-terrain-small.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      maxzoom: 6
+  - id: "terrain-medium"
+    name: "terrain-medium"
+    class: "terrain"
+    geometry: "raster"
+    <<: *extents84
+    Datasource:
+      file: "data/height/mixed-terrain-medium.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 7
+      maxzoom: 8
+  - id: "terrain-big"
+    name: "terrain-big"
+    class: "terrain"
+    geometry: "raster"
+    <<: *extents84
+    Datasource:
+      file: "data/height/mixed-terrain.vrt"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 8
   - id: builtup
     name: builtup
     geometry: polygon
@@ -166,6 +200,98 @@
         ) AS landcover_line
     properties:
       minzoom: 14
+  - id: "shade-small"
+    name: "shade-small"
+    class: "shade-over"
+    geometry: "raster"
+    <<: *extents84
+    Datasource:
+      file: "data/height/mixed-hillshade-small.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      maxzoom: 6
+  - id: "shade-medium-over"
+    name: "shade-medium-over"
+    class: "shade-over"
+    geometry: "raster"
+    <<: *extents84
+    Datasource:
+      file: "data/height/mixed-hillshade-medium.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 7
+      maxzoom: 7
+  - id: "shade-medium-overlay"
+    name: "shade-medium-overlay"
+    class: "shade-overlay"
+    geometry: "raster"
+    <<: *extents84
+    Datasource:
+      file: "data/height/mixed-hillshade-medium.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 8
+      maxzoom: 10
+  - id: "shade-big"
+    name: "shade-big"
+    class: "shade-overlay"
+    geometry: "raster"
+    <<: *extents84
+    Datasource:
+      file: "data/height/mixed-hillshade.vrt"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 9
+  - id: "slope-small"
+    name: "slope-small"
+    class: "slope-over"
+    geometry: "raster"
+    <<: *extents84
+    Datasource:
+      file: "data/height/mixed-slopeshade-small.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      maxzoom: 6
+  - id: "slope-medium-over"
+    name: "slope-medium-over"
+    class: "slope-over"
+    geometry: "raster"
+    <<: *extents84
+    Datasource:
+      file: "data/height/mixed-slopeshade-medium.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 7
+      maxzoom: 7
+  - id: "slope-medium-overlay"
+    name: "slope-medium-overlay"
+    class: "slope-overlay"
+    geometry: "raster"
+    <<: *extents84
+    Datasource:
+      file: "data/height/mixed-slopeshade-medium.tif"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 8
+      maxzoom: 8
+  - id: "slope-big"
+    name: "slope-big"
+    class: "slope-overlay"
+    geometry: "raster"
+    <<: *extents84
+    Datasource:
+      file: "data/height/mixed-slopeshade.vrt"
+      type: "gdal"
+    advanced: {}
+    properties:
+      minzoom: 9
   - id: water-lines-casing
     name: water-lines-casing
     geometry: linestring
